{\rtf1\ansi\ansicpg1252\cocoartf2707
\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\froman\fcharset0 Times-Roman;\f1\froman\fcharset0 Times-Bold;\f2\froman\fcharset0 Times-Italic;
\f3\fnil\fcharset0 LucidaGrande;}
{\colortbl;\red255\green255\blue255;\red14\green14\blue14;\red0\green0\blue233;\red67\green67\blue67;
}
{\*\expandedcolortbl;;\cssrgb\c6667\c6667\c6667;\cssrgb\c0\c0\c93333;\cssrgb\c33333\c33333\c33333;
}
{\*\listtable{\list\listtemplateid1\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid1}
{\list\listtemplateid2\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid101\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid2}
{\list\listtemplateid3\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid201\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid3}
{\list\listtemplateid4\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid301\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid4}
{\list\listtemplateid5\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid401\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid5}
{\list\listtemplateid6\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid501\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid6}
{\list\listtemplateid7\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid601\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid7}
{\list\listtemplateid8\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid701\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid8}
{\list\listtemplateid9\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid801\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid9}
{\list\listtemplateid10\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid901\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid10}
{\list\listtemplateid11\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1001\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid11}
{\list\listtemplateid12\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1101\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid12}
{\list\listtemplateid13\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1201\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid13}
{\list\listtemplateid14\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1301\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid14}
{\list\listtemplateid15\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1401\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid15}
{\list\listtemplateid16\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1501\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid16}
{\list\listtemplateid17\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1601\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid17}
{\list\listtemplateid18\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1701\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid18}
{\list\listtemplateid19\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1801\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid19}
{\list\listtemplateid20\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid1901\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid20}
{\list\listtemplateid21\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid2001\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid21}
{\list\listtemplateid22\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid2101\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{circle\}}{\leveltext\leveltemplateid2102\'01\uc0\u9702 ;}{\levelnumbers;}\fi-360\li1440\lin1440 }{\listname ;}\listid22}}
{\*\listoverridetable{\listoverride\listid1\listoverridecount0\ls1}{\listoverride\listid2\listoverridecount0\ls2}{\listoverride\listid3\listoverridecount0\ls3}{\listoverride\listid4\listoverridecount0\ls4}{\listoverride\listid5\listoverridecount0\ls5}{\listoverride\listid6\listoverridecount0\ls6}{\listoverride\listid7\listoverridecount0\ls7}{\listoverride\listid8\listoverridecount0\ls8}{\listoverride\listid9\listoverridecount0\ls9}{\listoverride\listid10\listoverridecount0\ls10}{\listoverride\listid11\listoverridecount0\ls11}{\listoverride\listid12\listoverridecount0\ls12}{\listoverride\listid13\listoverridecount0\ls13}{\listoverride\listid14\listoverridecount0\ls14}{\listoverride\listid15\listoverridecount0\ls15}{\listoverride\listid16\listoverridecount0\ls16}{\listoverride\listid17\listoverridecount0\ls17}{\listoverride\listid18\listoverridecount0\ls18}{\listoverride\listid19\listoverridecount0\ls19}{\listoverride\listid20\listoverridecount0\ls20}{\listoverride\listid21\listoverridecount0\ls21}{\listoverride\listid22\listoverridecount0\ls22}}
\margl1440\margr1440\vieww17400\viewh13920\viewkind0
\deftab720
\pard\pardeftab720\partightenfactor0

\f0\fs32 \cf2 \expnd0\expndtw0\kerning0
\'a0\
\pard\pardeftab720\partightenfactor0

\f1\b\fs64 \cf2 Jie Ren\
\'a0{{\NeXTGraphic citations.jpg \width5120 \height5100 \appleattachmentpadding0 \appleembedtype0 \appleaqc
}Â¬}\
\pard\pardeftab720\partightenfactor0

\f0\b0\fs32 \cf2 Computational Biology and Bioinformatics; Machine learning; Statistical Modeling\
\pard\pardeftab720\partightenfactor0
{\field{\*\fldinst{HYPERLINK "https://twitter.com/jessierenjie"}}{\fldrslt \cf3 \ul \ulc3 Twitter}} {\field{\*\fldinst{HYPERLINK "https://github.com/jessieren"}}{\fldrslt \cf3 \ul \ulc3 Github}} {\field{\*\fldinst{HYPERLINK "https://scholar.google.com/citations?user=Os9wmpkAAAAJ&hl=en"}}{\fldrslt \cf3 \ul \ulc3 Google Scholar}} \ul Email\ulnone \
\pard\pardeftab720\partightenfactor0
\cf4 \'a0\
\pard\pardeftab720\partightenfactor0
\cf2 I am a Senior Research Scientist in {\field{\*\fldinst{HYPERLINK "https://research.google/teams/brain/"}}{\fldrslt \cf3 \ul \ulc3 Google Research Brain Team}}. Previously, I received my PhD in {\field{\*\fldinst{HYPERLINK "https://www.qcb-dornsife.usc.edu/"}}{\fldrslt \cf3 \ul \ulc3 Computational Biology and Bioinformatics}} and MSc in Statistics at the University of Southern California. I am interested in developing robust and reliable methods for biomedical data analysis, with a focus on uncertainty quantification and out-of-distribution (OOD) detection in deep learning.\
\'a0\
\pard\pardeftab720\sa398\partightenfactor0

\f1\b\fs48 \cf2 News\
\pard\tx220\tx720\pardeftab720\li720\fi-720\partightenfactor0
\ls1\ilvl0
\f0\b0\fs32 \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}12/2023: Our study on selective generation for large language models, "Self-evaluation improves selective generation in large language models\'94, will be presented (spotlight) at NeurIPS 2023 I Can\'92t Believe It\'92s Not Better! (ICBINB) Workshop [{\field{\*\fldinst{HYPERLINK "https://openreview.net/forum?id=OptKBWmreP"}}{\fldrslt paper}}].
\f1\b\fs48 \expnd0\expndtw0\kerning0
\
\ls1\ilvl0
\f0\b0\fs32 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
07/2023: Our work on prompt selection and prompt ensemble for multimodal models, "A simple zero-shot prompt weighting technique to improve prompt ensembling in text-image models", will be presented at ICML 2023. [{\field{\*\fldinst{HYPERLINK "https://proceedings.mlr.press/v202/allingham23a.html"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0[{\field{\*\fldinst{HYPERLINK "https://twitter.com/JamesAllingham/status/1684125147756888064"}}{\fldrslt \cf3 \ul \ulc3 twitter]}}\'a0[{\field{\*\fldinst{HYPERLINK "https://github.com/google/uncertainty-baselines/tree/main/experimental/multimodal"}}{\fldrslt \cf3 \ul \ulc3 code]}}\'a0.\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
06/2023: Our paper "Improving Zero-shot Generalization and Robustness of Multi-modal Models" is accepted by CVPR 2023. Congratulations to Yunhao Ge. [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/pdf/2212.01758.pdf"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0[{\field{\*\fldinst{HYPERLINK "https://twitter.com/GeYunhao/status/1664478907012366337"}}{\fldrslt \cf3 \ul \ulc3 twitter]}}\'a0[{\field{\*\fldinst{HYPERLINK "https://github.com/gyhandy/Hierarchy-CLIP"}}{\fldrslt \cf3 \ul \ulc3 code]}}\'a0.\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
05/2023: Our paper "Out-of-Distribution Detection and Selective Generation for Conditional Language Models" is accepted (spotlight) by ICLR 2023. [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2209.15558"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0[{\field{\*\fldinst{HYPERLINK "https://twitter.com/peterjliu/status/1620157580114030592"}}{\fldrslt \cf3 \ul \ulc3 twitter]}}\'a0.\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
07/2022: Together with my colleagues at Google, we release a framework, Plex, for reliability in large pre-trained models. Please check out our latest blog post to learn more. [{\field{\*\fldinst{HYPERLINK "https://ai.googleblog.com/2022/07/towards-reliability-in-deep-learning.html"}}{\fldrslt \cf3 \ul \ulc3 blog]}}\'a0.\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
01/2022: Our new AI blog post is out :) It walks through our recent paper on enabling dermatology AI systems to detect the presence of conditinos unseen in training [{\field{\*\fldinst{HYPERLINK "https://ai.googleblog.com/2022/01/does-your-medical-image-classifier-know.html"}}{\fldrslt \cf3 \ul \ulc3 blog]}}\'a0.\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
12/2021: Our paper "Exploring the Limits of Out-of-Distribution Detection" [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2106.03004"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0will be presented at NeurIPS 2021.\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
07/2021: Together with {\field{\*\fldinst{HYPERLINK "https://polkirichenko.github.io/"}}{\fldrslt \cf3 \ul \ulc3 Polina Kirichenko}}, {\field{\*\fldinst{HYPERLINK "https://scholar.google.com/citations?user=-SjNMu8AAAAJ&hl=en"}}{\fldrslt \cf3 \ul \ulc3 Haleh Akrami}}, {\field{\*\fldinst{HYPERLINK "http://pages.cs.wisc.edu/~sharonli/"}}{\fldrslt \cf3 \ul \ulc3 Sharon Yixuan Li}} and {\field{\*\fldinst{HYPERLINK "https://scholar.google.com/citations?user=D_CuHyUAAAAJ&hl=en"}}{\fldrslt \cf3 \ul \ulc3 Serg\'fcl Ayd\'f6re}}, we are going to lead a {\field{\*\fldinst{HYPERLINK "https://icml.cc/virtual/2021/affinityworkshop/10849"}}{\fldrslt \cf3 \ul \ulc3 breakout session}} at July 21 7:25 pm ET at WiML Un-Workshop at ICML 2021 on "Does your model know what it doesn\'92t know? Uncertainty estimation and OOD detection in deep learning."\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
07/2021: Our paper "A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection" [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2106.09022"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0[{\field{\*\fldinst{HYPERLINK "https://twitter.com/jessierenjie/status/1418321220546613248"}}{\fldrslt \cf3 \ul \ulc3 poster]}}\'a0will be presented at ICML 2021 UDL workshop.\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
07/2021: Our paper "Exploring the Limits of Out-of-Distribution Detection" [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2106.03004"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0will be presented at ICML 2021 UDL workshop.\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
12/2019: We will present our work of "Likelihood Ratios for Out-of-Distribution Detection" at NeurIPS 2019. [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/1906.02845"}}{\fldrslt \cf3 \ul \ulc3 paper}}][{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/1906.02845"}}{\fldrslt \cf3 \ul \ulc3 paper] [}}{\field{\*\fldinst{HYPERLINK "https://github.com/google-research/google-research/tree/master/genomics_ood"}}{\fldrslt \cf3 \ul \ulc3 code] [}}{\field{\*\fldinst{HYPERLINK "http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-llr-ood-poster.pdf"}}{\fldrslt \cf3 \ul \ulc3 poster] [}}{\field{\*\fldinst{HYPERLINK "https://www.youtube.com/watch?v=-FduW9ZWAR4"}}{\fldrslt \cf3 \ul \ulc3 3-minute video]\'a0}}[{\field{\*\fldinst{HYPERLINK "https://ai.googleblog.com/2019/12/improving-out-of-distribution-detection.html"}}{\fldrslt \cf3 \ul \ulc3 blog]}}\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
06/2019: I am joining Google Brain as a Research Scientist.\'a0\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
06/2018: I am joining Google Brain AI Residency program.\'a0\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
03/2018: Our work is highlighted in {\field{\*\fldinst{HYPERLINK "https://www.nature.com/articles/d41586-018-03358-3"}}{\fldrslt 
\f2\i \cf3 \ul \ulc3 Nature News }}titled "Machine learning spots treasure trove of elusive viruses Artificial intelligence could speed up metagenomic studies that look for species unknown to science.".\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
04/2017: I am honored to receive the prestigious {\field{\*\fldinst{HYPERLINK "https://twitter.com/uscbiosci/status/858070439435710464"}}{\fldrslt \cf3 \ul \ulc3 USC PhD Achievement Award}} (6 winners university-wide per year).\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
03/2017: I am honored to receive Women in Science and Engineering Travel Grant.\'a0\
\ls1\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
04/2016: I am honored to receive {\field{\*\fldinst{HYPERLINK "https://dornsife.usc.edu/cams/cams-prize/"}}{\fldrslt \cf3 \ul \ulc3 USC CAMS Graduate Student Prize}} for Excellence in Research with a Substantial Mathematical Component (2 winners university-wide per year).\'a0\
\pard\pardeftab720\sa320\partightenfactor0
\cf2 \'a0\
\pard\pardeftab720\sa398\partightenfactor0

\f1\b\fs48 \cf2 Publications\
\pard\pardeftab720\sa425\partightenfactor0

\fs32 \cf2 Out-of-Distribution Detection in Deep Learning
\f0\b0 \kerning1\expnd0\expndtw0 \
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls2\ilvl0
\f1\b \cf2 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Building One-class Detector for Anything: Open-vocabulary Zero-shot OOD Detection Using Text-image Models. 
\f0\b0 Presented at the 
\f2\i ICML workshop on Knowledge and Logical Reasoning in the Era of Data-driven Learning  
\f0\i0 (2023).\'a0 Yunhao Ge*, 
\f2\i Jie Ren*
\f0\i0 , Jiaping Zhao, Kaifeng Chen, Andrew Gallagher, Laurent Itti, Balaji Lakshminarayanan.[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2305.17207"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0 \'a0\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls3\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Out-of-Distribution Detection and Selective Generation for Conditional Language Models. 
\f2\i\b0 ICLR
\f0\i0  (2023).\'a0 
\f2\i Jie Ren
\f0\i0 , Jiaming Luo, Yao Zhao, Kundan Krishna, Mohammad Saleh, Balaji Lakshminarayanan, Peter J. Liu.[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2209.15558"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0 \'a0\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls4\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
A Simple Approach to Improve Single-Model Deep Uncertainty via Distance-Awareness. 
\f2\i\b0 Journal of Machine Learning Research
\f0\i0  23 (2022): 1-63.\'a0 Jeremiah Zhe Liu*, Shreyas Padhy*, 
\f2\i Jie Ren*
\f0\i0 , Zi Lin, Yeming Wen, Ghassen Jerfel, Zack Nado, Jasper Snoek, Dustin Tran, and Balaji Lakshminarayanan.[{\field{\*\fldinst{HYPERLINK "https://www.jmlr.org/papers/volume24/22-0479/22-0479.pdf"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0 \'a0\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls5\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
MLR-OOD: A Markov Chain Based Likelihood Ratio Method for Out-Of-Distribution Detection of Genomic Sequences. 
\f2\i\b0 Journal of Molecular Biology
\f0\i0  (2022).\'a0 Xin Bai, 
\f2\i Jie Ren
\f0\i0 , Fengzhu Sun.[{\field{\*\fldinst{HYPERLINK "https://www.sciencedirect.com/science/article/pii/S0022283622001668"}}{\fldrslt \cf3 \ul \ulc3 paper]}}[{\field{\*\fldinst{HYPERLINK "https://docs.google.com/presentation/d/1XmsWPViWmXh0cwOBFRXlBE2CnsWyaynzN9Qrmv2pNRA/edit?usp=sharing"}}{\fldrslt \cf3 \ul \ulc3 slides]}} \'a0\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls6\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Exploring the Limits of Out-of-Distribution Detection. 
\f2\i\b0 NeurIPS
\f0\i0  (2021).\'a0 Stanislav Fort*, 
\f2\i Jie Ren*
\f0\i0 , and Balaji Lakshminarayanan.[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2106.03004"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0 \'a0\
\pard\tx220\tx720\pardeftab720\li720\fi-720\partightenfactor0
\ls7\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection. \'a0
\f0\b0 Presented at the 
\f2\i ICML workshop on Uncertainty and Robustness in Deep Learning
\f0\i0  (2021).\'a0 
\f2\i Jie Ren
\f0\i0 , Stanislav Fort, Jeremiah Liu, Abhijit Guha Roy, Shreyas Padhy, and Balaji Lakshminarayanan. [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2106.09022"}}{\fldrslt \cf3 \ul \ulc3 paper]\'a0}}{\field{\*\fldinst{HYPERLINK "https://twitter.com/jessierenjie/status/1418321220546613248"}}{\fldrslt \cf3 \ul \ulc3 [poster]}}\cf3 \ul \ulc3  [{\field{\*\fldinst{HYPERLINK "https://github.com/google/uncertainty-baselines/blob/main/experimental/near_ood/vit/ood_utils.py"}}{\fldrslt code}}]\cf2 \ulnone \
\pard\tx720\pardeftab720\partightenfactor0
\cf2 \
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls8\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Does Your Dermatology Classifier Know What It Doesn't Know? Detecting the Long-Tail of Unseen Conditions.
\f0\b0 \'a0
\f2\i Medical Image Analysis
\f0\i0 \'a075 (2022): 102274. Abhijit Guha Roy*, 
\f2\i Jie Ren*
\f0\i0 , Shekoofeh Azizi, Aaron Loh, Vivek Natarajan, Basil Mustafa, Nick Pawlowski et al.\'a0[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2104.03829"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0[{\field{\*\fldinst{HYPERLINK "https://ai.googleblog.com/2022/01/does-your-medical-image-classifier-know.html"}}{\fldrslt \cf3 \ul \ulc3 blog}}]\
\ls8\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Revisiting One-vs-All Classifiers for Predictive Uncertainty and Out-of-Distribution Detection in Neural Networks. \'a0
\f0\b0 Presented at the 
\f2\i ICML workshop on Uncertainty and Robustness in Deep Learning
\f0\i0  (2020).\'a0 Shreyas Padhy, Zachary Nado, 
\f2\i Jie Ren,
\f0\i0  Jeremiah Liu, Jasper Snoek, and Balaji Lakshminarayanan. [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2007.05134"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0
\f1\b  \'a0 \'a0
\f0\b0 \
\ls8\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Likelihood Ratios for Out-of-Distribution Detection. 
\f2\i\b0 NeurIPS
\f0\i0  (2019).\'a0
\f2\i Jie Ren
\f0\i0 , Peter J. Liu, Emily Fertig, Jasper Snoek, Ryan Poplin, Mark A. DePristo, Joshua V. Dillon, and Balaji Lakshminarayanan.\'a0[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/1906.02845"}}{\fldrslt \cf3 \ul \ulc3 paper] [}}{\field{\*\fldinst{HYPERLINK "https://github.com/google-research/google-research/tree/master/genomics_ood"}}{\fldrslt \cf3 \ul \ulc3 code] [}}{\field{\*\fldinst{HYPERLINK "http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-llr-ood-poster.pdf"}}{\fldrslt \cf3 \ul \ulc3 poster] [}}{\field{\*\fldinst{HYPERLINK "https://www.youtube.com/watch?v=-FduW9ZWAR4"}}{\fldrslt \cf3 \ul \ulc3 3-minute video]\'a0}}[{\field{\*\fldinst{HYPERLINK "https://ai.googleblog.com/2019/12/improving-out-of-distribution-detection.html"}}{\fldrslt \cf3 \ul \ulc3 blog]}} \'a0\
\pard\pardeftab720\sa425\partightenfactor0

\f1\b \cf2 \
Uncertainty and Robustness in Deep Learning\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls9\ilvl0\cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}Universal Self-Consistency for Large Language Model Generation.
\f0\b0  arXiv preprint arXiv:2311.17311 (2023). Chen, Xinyun, Renat Aksitov, Uri Alon, 
\f2\i Jie Ren
\f0\i0 , Kefan Xiao, Pengcheng Yin, Sushant Prakash, Charles Sutton, Xuezhi Wang, and Denny Zhou. 
\f1\b \
{\listtext	\uc0\u8226 	}On Uncertainty Calibration and Selective Generation in Probabilistic Neural Summarization: A Benchmark Study. 
\f2\i\b0 EMNLP
\f0\i0  (2023). Polina Zablotskaia, Du Phan, Joshua Maynez, Shashi Narayan, 
\f2\i Jie Ren
\f0\i0 , Jeremiah Liu. [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2304.08653"}}{\fldrslt paper}}]\
\ls9\ilvl0
\f2\i {\listtext	\uc0\u8226 	}
\f1\i0\b \expnd0\expndtw0\kerning0
Improving the Robustness of Summarization Models by Detecting and Removing Input Noise. 
\f2\i\b0 EMNLP
\f0\i0  (2023).\'a0 Kundan Krishna, Yao Zhao, 
\f2\i Jie Ren
\f0\i0 , Balaji Lakshminarayanan, Jiaming Luo, Mohammad Saleh, Peter J. Liu.[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2212.09928"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0 \'a0
\f2\i \
\ls9\ilvl0
\f1\i0\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
A Simple Zero-shot Prompt Weighting Technique to Improve Prompt Ensembling in Text-Image Models. 
\f2\i\b0 ICML
\f0\i0  (2023).\'a0 James Urquhart Allingham*, 
\f2\i Jie Ren*
\f0\i0 , Michael W Dusenberry, Jeremiah Zhe Liu, Xiuye Gu, Yin Cui, Dustin Tran, Balaji Lakshminarayanan.[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2302.06235"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0 
\f2\i \kerning1\expnd0\expndtw0 \
\ls9\ilvl0
\f0\i0 {\listtext	\uc0\u8226 	}
\f1\b Morse Neural Networks for Uncertainty Quantification.
\f0\b0  Presented at
\f2\i  the ICML workshop on
\f0\i0  
\f2\i Structured Probabilistic Inference and Generative Modeling 
\f0\i0 Benoit Dherin, Huiyi Hu, 
\f2\i Jie Ren
\f0\i0 , Michael W. Dusenberry, and Balaji Lakshminarayanan.[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2307.00667"}}{\fldrslt \cf2 paper}}]\expnd0\expndtw0\kerning0
\'a0\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls10\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Improving Zero-shot Generalization and Robustness of Multi-modal Models. 
\f2\i\b0 CVPR
\f0\i0  (2022).\'a0 Yunhao Ge*, 
\f2\i Jie Ren*
\f0\i0 , Yuxiao Wang, Andrew Gallagher, Ming-Hsuan Yang, Laurent Itti, Hartwig Adam, Balaji Lakshminarayanan, Jiaping Zhao.[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2212.01758"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0 \'a0\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls11\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Plex: Towards reliability using pretrained large model extensions. 
\f2\i\b0 arXiv preprint arXiv:2207.07411
\f0\i0  (2022).\'a0 Dustin Tran, Jeremiah Liu, Michael W Dusenberry, Du Phan, Mark Collier, 
\f2\i Jie Ren
\f0\i0 , Kehang Han, Zi Wang, Zelda Mariet, Huiyi Hu, Neil Band, Tim GJ Rudner, Karan Singhal, Zachary Nado, Joost van Amersfoort, Andreas Kirsch, Rodolphe Jenatton, Nithum Thain, Honglin Yuan, Kelly Buchanan, Kevin Murphy, D Sculley, Yarin Gal, Zoubin Ghahramani, Jasper Snoek, Balaji Lakshminarayanan.[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2207.07411"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0 \'a0\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls12\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Uncertainty Baselines: Benchmarks for Uncertainty & Robustness in Deep Learning.\'a0
\f0\b0 Presented at the 
\f2\i NeurIPS workshop on Bayesian deep learning
\f0\i0 \'a0(2021). Zachary Nado, Neil Band, Mark Collier, Josip Djolonga, Michael W. Dusenberry, Sebastian Farquhar, Angelos Filos et al.\'a0[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/2106.04015"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls13\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
SummAE: Summae: Zero-Shot Abstractive Text Summarization Using Length-Agnostic Auto-Encoders.
\f0\b0 \'a0
\f2\i arXiv preprint arXiv:1910.00998.
\f0\i0  Peter J. Liu, Yu-An Chung, and\'a0
\f2\i Jie Ren. [{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/1910.00998"}}{\fldrslt \cf3 \ul \ulc3 paper}}]
\f0\i0 \
\ls13\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Can You Trust Your Model\'92s Uncertainty? Evaluating Predictive Uncertainty Under Dataset Shift. 
\f2\i\b0 NeurIPS
\f0\i0  (2019).\'a0 Yaniv Ovadia, Emily Fertig, 
\f2\i Jie Ren
\f0\i0 , Zachary Nado, David Sculley, Sebastian Nowozin, Joshua V. Dillon, Balaji Lakshminarayanan, and Jasper Snoek.\'a0[{\field{\*\fldinst{HYPERLINK "https://arxiv.org/abs/1906.02530"}}{\fldrslt \cf3 \ul \ulc3 paper}}] [{\field{\*\fldinst{HYPERLINK "https://github.com/google-research/google-research/tree/master/uq_benchmark_2019"}}{\fldrslt \cf3 \ul \ulc3 code}}] [{\field{\*\fldinst{HYPERLINK "http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-benchmark-poster.pdf"}}{\fldrslt \cf3 \ul \ulc3 poster}}] [{\field{\*\fldinst{HYPERLINK "https://ai.googleblog.com/2020/01/can-you-trust-your-models-uncertainty.html"}}{\fldrslt \cf3 \ul \ulc3 blog}}]\'a0 \'a0 \'a0\
\pard\pardeftab720\sa320\partightenfactor0
\cf2 \'a0\
\pard\pardeftab720\sa425\partightenfactor0

\f1\b \cf2 Machine Learning in Biological Sciences\
\pard\tx220\tx720\pardeftab720\li720\fi-720\partightenfactor0
\ls14\ilvl0\cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
KIMI: Knockoff Inference for Motif Identification From Molecular Sequences With Controlled False Discovery Rate
\f0\b0 . 
\f2\i Bioinformatics
\f0\i0 \'a037, no. 6 (2021): 759-766. Bai Xin, 
\f2\i Jie Ren
\f0\i0 , Yingying Fan, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://academic.oup.com/bioinformatics/article-abstract/37/6/759/5942973"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0\
\pard\tx720\pardeftab720\partightenfactor0
\cf2 \
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls15\ilvl0
\f1\b \cf2 Identifying Viruses From Metagenomic Data Using Deep Learning.\'a0
\f2\i\b0 Quantitative Biology 
\f0\i0 (2020): 1-14
\f2\i .\'a0Jie Ren*, 
\f0\i0 Kai Song*, Chao Deng, Nathan A. Ahlgren, Jed A. Fuhrman, Yi Li, Xiaohui Xie, Ryan Poplin, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://link.springer.com/content/pdf/10.1007/s40484-019-0187-4.pdf"}}{\fldrslt \cf3 \ul \ulc3 paper}}] [{\field{\*\fldinst{HYPERLINK "https://github.com/jessieren/DeepVirFinder"}}{\fldrslt \cf3 \ul \ulc3 code}}]\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls16\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
A Network-Based Integrated Framework for Predicting Virus\'96Prokaryote Interactions.
\f0\b0 \'a0
\f2\i NAR Genomics and Bioinformatics
\f0\i0 \'a02, no. 2 (2020): lqaa044. Weili Wang*, 
\f2\i Jie Ren*
\f0\i0 , Kujin Tang, Emily Dart, Julio Cesar Ignacio-Espinoza, Jed A. Fuhrman, Jonathan Braun, Fengzhu Sun, and Nathan A. Ahlgren.\'a0[{\field{\*\fldinst{HYPERLINK "https://academic.oup.com/nargab/article/2/2/lqaa044/5861484?login=true"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0[{\field{\*\fldinst{HYPERLINK "https://github.com/jessieren/VirHostMatcher"}}{\fldrslt \cf3 \ul \ulc3 code}}]\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls17\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Rapid Diagnosis and Comprehensive Bacteria Profiling of Sepsis Based on Cell-Free Dna.\'a0
\f2\i\b0 Journal of Translational Medicine
\f0\i0 \'a018, no. 1 (2020): 1-10. Pei Chen, Shuo Li, Wenyuan Li, 
\f2\i Jie Ren
\f0\i0 , Fengzhu Sun, Rui Liu, and Xianghong Jasmine Zhou. [{\field{\*\fldinst{HYPERLINK "https://link.springer.com/article/10.1186/s12967-019-02186-x;"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0\
\ls17\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Afann: Bias Adjustment for Alignment-Free Sequence Comparison Based on Sequencing Data Using Neural Network Regression.
\f0\b0 \'a0
\f2\i Genome Biology
\f0\i0  20.1 (2019): 1-17
\f2\i .\'a0
\f0\i0 Kujin Tang,
\f2\i  Jie Ren
\f0\i0 , and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1872-3"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls17\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Metagenomic Unmapped Reads Provide Important Insights Into Human Microbiota and Disease Associations.\'a0
\f2\i\b0 Genome Biology
\f0\i0 \'a020, no. 1 (2019): 1-13. Zifan Zhu,
\f2\i  Jie Ren
\f0\i0 , Sonia Michail, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1773-5"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls17\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Alignment-Free Oligonucleotide Frequency Dissimilarity Measure Improves Prediction of Hosts From Metagenomically-Derived Viral Sequences.\'a0
\f2\i\b0 Nucleic Acids Research 
\f0\i0 45.1 (2018), 39-53
\f2\i .\'a0
\f0\i0 Nathan A. Ahlgren*, 
\f2\i Jie Ren*
\f0\i0 , Yang Y. Lu, Jed A. Fuhrman, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://academic.oup.com/nar/article/45/1/39/2605663?login=true"}}{\fldrslt \cf3 \ul \ulc3 paper}}] [{\field{\*\fldinst{HYPERLINK "https://github.com/jessieren/VirHostMatcher"}}{\fldrslt \cf3 \ul \ulc3 code}}]\
\ls17\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
VirFinder: A Novel K-Mer Based Tool for Identifying Viral Sequences From Assembled Metagenomic Data.\'a0
\f2\i\b0 Microbiome
\f0\i0 \'a05, no. 1 (2017): 1-20. 
\f2\i Jie Ren*,\'a0
\f0\i0 Nathan A. Ahlgren*, Yang Young Lu, Jed A. Fuhrman, and Fengzhu Sun.\'a0[{\field{\*\fldinst{HYPERLINK "https://link.springer.com/article/10.1186/s40168-017-0283-5"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0[{\field{\*\fldinst{HYPERLINK "https://github.com/jessieren/VirFinder"}}{\fldrslt \cf3 \ul \ulc3 code}}] [{\field{\*\fldinst{HYPERLINK "https://anaconda.org/bioconda/r-virfinder"}}{\fldrslt \cf3 \ul \ulc3 R package}}] [{\field{\*\fldinst{HYPERLINK "https://www.nature.com/articles/d41586-018-03358-3"}}{\fldrslt \cf3 \ul \ulc3 news}}]\
\ls17\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Identifying Group-Specific Sequences for Microbial Communities Using Long K-Mer Sequence Signatures. 
\f2\i\b0 Frontiers in Microbiology
\f0\i0 \'a09 (2018): 872. Ying Wang, Lei Fu, 
\f2\i Jie Ren
\f0\i0 , Zhaoxia Yu, Ting Chen, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://www.frontiersin.org/articles/10.3389/fmicb.2018.00872/full"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\pard\tx220\tx720\pardeftab720\li720\fi-720\partightenfactor0
\ls17\ilvl0\cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}
\f1\b \expnd0\expndtw0\kerning0
Prediction of Virus-Host Infectious Association by Supervised Learning Methods. 
\f2\i\b0 BMC Bioinformatics
\f0\i0  18.3 (2017): 143-154. Mengge Zhang, Lianping Yang, 
\f2\i Jie Ren
\f0\i0 , Nathan A. Ahlgren, Jed A. Fuhrman, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1473-7"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\uc0\u8232 \
\pard\tx720\pardeftab720\sa320\partightenfactor0
\cf2 \
\pard\pardeftab720\sa425\partightenfactor0

\f1\b \cf2 Statistical Modeling in Genomics Data\kerning1\expnd0\expndtw0 \
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa425\partightenfactor0
\ls18\ilvl0\cf2 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
A New Context Tree Inference Algorithm for Variable Length Markov Chain Model with Applications to Biological Sequence Analyses.
\f0\b0  
\f2\i Journal of Computational Biology
\f0\i0 \'a0(2022). Shaokun An, 
\f2\i Jie Ren
\f0\i0 , Fengzhu Sun, and Lin Wan. [{\field{\*\fldinst{HYPERLINK "https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0604"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls19\ilvl0
\f1\b \cf2 Confidence Intervals for Markov Chain Transition Probabilities Based on Next Generation Sequencing Reads Data.
\f0\b0 \'a0
\f2\i Quantitative Biology 
\f0\i0 8 (2020): 143-154
\f2\i .\'a0
\f0\i0 Lin Wan, Xin Kang, 
\f2\i Jie Ren
\f0\i0 , and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://link.springer.com/article/10.1007/s40484-020-0200-y"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\'a0\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa320\partightenfactor0
\ls19\ilvl0
\f1\b \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Predicting the Number of Bases to Attain Sufficient Coverage in High-Throughput Sequencing Experiments. 
\f2\i\b0 Journal of Computational Biology
\f0\i0 \'a027, no. 7 (2020): 1130-1143. Chao Deng, Timothy Daley, Peter Calabrese, 
\f2\i Jie Ren
\f0\i0 , and Andrew D. Smith.\'a0[{\field{\*\fldinst{HYPERLINK "https://www.liebertpub.com/doi/abs/10.1089/cmb.2019.0264"}}{\fldrslt \cf3 \ul \ulc3 paper]}}\'a0\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Reads Binning Improves Alignment-Free Metagenome Comparison.
\f0\b0  
\f2\i Frontiers in Genetics
\f0\i0 \'a010 (2019): 1156. Kai Song, 
\f2\i Jie Ren
\f0\i0 , and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://internal-journal.frontiersin.org/articles/10.3389/fgene.2019.01156/full"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Alignment-Free Sequence Analysis and Applications.\'a0
\f2\i\b0 Annual Review of Biomedical Data Science
\f0\i0 \'a01 (2018): 93-114. 
\f2\i Jie Ren
\f0\i0 , Xin Bai, Yang Young Lu, Kujin Tang, Ying Wang, Gesine Reinert, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://www.annualreviews.org/doi/abs/10.1146/annurev-biodatasci-080917-013431"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Alignment-Free Genome Comparison Enables Accurate Geographic Sourcing of White Oak DNA.\'a0
\f2\i\b0 BMC Genomics
\f0\i0 \'a019, no. 1 (2018): 1-16. Kujin Tang, 
\f2\i Jie Ren
\f0\i0 , Richard Cronn, David L. Erickson, Brook G. Milligan, Meaghan Parker-Forney, John L. Spouge, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5253-1"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Gut Microbial and Metabolomic Profiles After Fecal Microbiota Transplantation in Pediatric Ulcerative Colitis Patients.
\f0\b0 \'a0
\f2\i FEMS Microbiology Ecology
\f0\i0 \'a094, no. 9 (2018): fiy133.\'a0Nusbaum, David J., Fengzhu Sun,
\f2\i  Jie Ren
\f0\i0 , Zifan Zhu, Natalie Ramsy, Nicholas Pervolarakis, Sachin Kunde et al. [{\field{\*\fldinst{HYPERLINK "https://academic.oup.com/femsec/article/94/9/fiy133/5053801?login=true"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Optimal Choice of Word Length When Comparing Two Markov Sequences Using a \uc0\u967 2-Statistic.\'a0
\f2\i\b0 BMC Genomics
\f0\i0  18.6 (2017): 19-30.\'a0Xin Bai, Kujin Tang, 
\f2\i Jie Ren
\f0\i0 , Michael Waterman, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://link.springer.com/article/10.1186/s12864-017-4020-z"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Cafe: Accelerated\'a0Alignment-Free Sequence Analysis. 
\f2\i\b0 Nucleic Acids Research
\f0\i0  45.1 (2017): W554-W559.\'a0Yang Y. Lu, Kujin Tang, Jie Ren, Jed A. Fuhrman, Michael S. Waterman, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://academic.oup.com/nar/article/45/W1/W554/3791208?login=true"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\'a0\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Alignment-Free Transcriptomic and Metatranscriptomic Comparison Using Sequencing Signatures With Variable Length Markov Chains.\'a0
\f2\i\b0 Scientific Reports
\f0\i0  6.1 (2016): 1-15. Weinan Liao*, 
\f2\i Jie Ren*
\f0\i0 , Kun Wang, Shun Wang, Feng Zeng, Ying Wang, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://www.nature.com/articles/srep37243"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Inference of Markovian Properties of Molecular Sequences From Ngs Data and Applications to Comparative Genomics.\'a0
\f2\i\b0 Bioinformatics 
\f0\i0 32.7 (2016): 993-1000.\'a0
\f2\i Jie Ren
\f0\i0 , Kai Song, Minghua Deng, Gesine Reinert, Charles H. Cannon, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://academic.oup.com/bioinformatics/article/32/7/993/2240327?login=true"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
New Developments of Alignment-Free Sequence Comparison: Measures, Statistics and Next-Generation Sequencing.\'a0
\f2\i\b0 Briefings in Bioinformatics
\f0\i0  15.3 (2014): 343-353.\'a0Kai Song, 
\f2\i Jie Ren
\f0\i0 , Gesine Reinert, Minghua Deng, Michael S. Waterman, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://academic.oup.com/bib/article/15/3/343/182355?login=true"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Multiple Alignment-Free Sequence Comparison.\'a0
\f2\i\b0 Bioinformatics 29.21 (2013): 2690-2698.\'a0Jie Ren
\f0\i0 , Kai Song, Fengzhu Sun, Minghua Deng, and Gesine Reinert. [{\field{\*\fldinst{HYPERLINK "https://academic.oup.com/bioinformatics/article/29/21/2690/195361?login=true"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Alignment-Free Sequence Comparison Based on Next-Generation Sequencing Reads.\'a0
\f2\i\b0 Journal of Computational Biology
\f0\i0  20.2 (2013): 64-79. Kai Song, 
\f2\i Jie Ren,
\f0\i0  Zhiyuan Zhai, Xuemei Liu, Minghua Deng, and Fengzhu Sun. [{\field{\*\fldinst{HYPERLINK "https://www.liebertpub.com/doi/abs/10.1089/cmb.2012.0228"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\ls19\ilvl0
\f1\b \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Comparison of Metagenomic Samples Using Sequence Signatures.\'a0
\f2\i\b0 BMC Genomics
\f0\i0  13.1 (2012): 1-17.
\f1\b \'a0
\f0\b0 Bai Jiang*, Kai Song*, 
\f2\i Jie Ren*
\f0\i0 , Minghua Deng, Fengzhu Sun, and Xuegong Zhang. [{\field{\*\fldinst{HYPERLINK "https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-13-730"}}{\fldrslt \cf3 \ul \ulc3 paper}}]\
\pard\pardeftab720\sa320\partightenfactor0
\cf2 \'a0\
\pard\pardeftab720\sa398\partightenfactor0

\f1\b\fs48 \cf2 Talks\
\pard\tx220\tx720\pardeftab720\li720\fi-720\partightenfactor0
\ls20\ilvl0
\f0\b0\fs32 \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2023 {\field{\*\fldinst{HYPERLINK "http://wids-santaclara.org/?i=1"}}{\fldrslt \cf3 \ul \ulc3 Women in Data Science SANTA CLARA 2023}}: Building neural networks that know what they don't know.\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2023 {\field{\*\fldinst{HYPERLINK "https://sites.google.com/view/vand-cvpr23/home"}}{\fldrslt \cf3 \ul \ulc3 Visual Anomaly and Novelty Detection Workshop}} @ CVPR 2023: OOD detection and uncertainty estimation using text-image models.\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2022 Amazon BioSciences Colloquium: Does Your Model Know What It Doesn\'92t Know? Exploring the Limits of Out-of-distribution Detection.\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2022 UCL Centre for Artificial Intelligence:\'a0{\field{\*\fldinst{HYPERLINK "https://www.youtube.com/watch?v=8cdBWK4Y45s"}}{\fldrslt \cf3 \ul \ulc3 Reliable Deep Anomaly Detection}}\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2021 Anomaly Detection for Scientific Discovery Seminars:\'a0{\field{\*\fldinst{HYPERLINK "https://ad4sd.github.io/seminars/"}}{\fldrslt \cf3 \ul \ulc3 Exploring the limits of out-of-distribution detection in vision and biomedical applications}}\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2021 ICML WiML Un-Workshop:\'a0{\field{\*\fldinst{HYPERLINK "https://docs.google.com/presentation/d/1E90g5yJpYZBa-_jRYGc0IMSnE0axf0wHskUKS1XD-9Y/edit?usp=sharing"}}{\fldrslt \cf3 \ul \ulc3 Uncertainty Estimation and Out-of-Distribution (Ood) Detection in Deep Learning}}\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2019 NeurIPS:\'a0{\field{\*\fldinst{HYPERLINK "https://www.youtube.com/watch?v=-FduW9ZWAR4"}}{\fldrslt \cf3 \ul \ulc3 Likelihood Ratios for Out-of-Distribution Detection.}}\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2016 Molecular and Computational Biology Retreat, University of Southern California: Predicting Virus-Host Interactions Using Sequence Signatures.\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2015 Joint USC/UCLA Bioinformatics Colloquium: Statistical Inference of Markov Models of Genomic Sequences Using NGS Data.\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2015 Research in Computational Molecular Biology (RECOMB)-Seq: Inference of Markovian Properties of Molecular Sequences From NGS Data and Applications to Comparative Genomics.\
\ls20\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
2013 Molecular and Computational Biology Retreat, University of Southern California: Alignment-Free Genome and Metagenome Comparison Based on Next Generation Sequencing Reads.\
\pard\pardeftab720\sa320\partightenfactor0
\cf2 \'a0\
\pard\pardeftab720\sa398\partightenfactor0

\f1\b\fs48 \cf2 Mentorship and Community\
\pard\tx220\tx720\pardeftab720\li720\fi-720\partightenfactor0
\ls21\ilvl0
\f0\b0\fs32 \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Interns: Yunhao Ge, James Allingham, Kundan Krishna, Yu-An Chung.\
\ls21\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Invited reviewer for machine learning venues: NeurIPS (2019-), ICML (2019-), ICLR (2019-), IEEE Transactions on Artificial Intelligence\'a0\
\ls21\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Invited reviewer for bioinformatics journals: Nature Microbiology, Cell Systems,\'a0Medical Image Analysis, Nucleic Acids Research, Gut, Bioinformatics, Giga Science, Scientific Reports, PLOS Computational Biolofy, PLOS ONE.\'a0\
\ls21\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Program Committee (PC) for ICML workshop on Uncertainty and Robustness in Deep Learning\'a0 (2020-2021) \
\pard\pardeftab720\sa320\partightenfactor0
\cf2 \'a0\
\pard\pardeftab720\sa398\partightenfactor0

\f1\b\fs48 \cf2 Education\
\pard\tx220\tx720\pardeftab720\li720\fi-720\partightenfactor0
\ls22\ilvl0
\f0\b0\fs32 \cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Postdoc Fellow, University of Southern California, Los Angeles, USA. (2017-2018).\
\ls22\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Ph.D. in Computational Biology and Bioinformatics, University of Southern California, Los Angeles, USA. (2013-2017).\
\pard\tx940\tx1440\pardeftab720\li1440\fi-1440\partightenfactor0
\ls22\ilvl1\cf2 \kerning1\expnd0\expndtw0 {\listtext	
\f3 \uc0\u9702 
\f0 	}\expnd0\expndtw0\kerning0
Advisors: {\field{\*\fldinst{HYPERLINK "https://dornsife.usc.edu/cf/faculty-and-staff/faculty.cfm?pid=1003742"}}{\fldrslt \cf3 \ul \ulc3 Fengzhu Sun}}, {\field{\*\fldinst{HYPERLINK "https://dornsife.usc.edu/cf/faculty-and-staff/faculty.cfm?pid=1003269"}}{\fldrslt \cf3 \ul \ulc3 Jed Fuhrman}}, {\field{\*\fldinst{HYPERLINK "https://dornsife.usc.edu/labs/msw/"}}{\fldrslt \cf3 \ul \ulc3 Michael Waterman}}, {\field{\*\fldinst{HYPERLINK "http://www.stats.ox.ac.uk/~reinert/"}}{\fldrslt \cf3 \ul \ulc3 Gesine Reinert}}\
\pard\tx220\tx720\pardeftab720\li720\fi-720\partightenfactor0
\ls22\ilvl0\cf2 \kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
M.S. in Statistics, University of Southern California, Los Angeles, USA. (2016-2017).\
\ls22\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Exchange Student in Bioinformatics, Stockholm University, Stockholm, Sweden. (2011).\
\ls22\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
M.S. in Probability and Statistics, Peking University, Beijing, China. (2010-2013).\
\ls22\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Double Major in Economics, Peking University, Beijing, China. (2009-2011).\
\ls22\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
Exchange Student in Mathematics, Hong Kong Baptist University, Hong Kong SAR. (2008).\
\ls22\ilvl0\kerning1\expnd0\expndtw0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
B.S. in Information and Computing Science, Minzu University, Beijing, China. (2006-2010).\
\pard\pardeftab720\sa320\partightenfactor0
\cf2 \'a0\
\'a0\
}
