<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">.lst-kix_6ftjd98kskfz-5>li:before{content:"\0025a0   "}.lst-kix_6ftjd98kskfz-6>li:before{content:"\0025cf   "}.lst-kix_6ftjd98kskfz-7>li:before{content:"\0025cb   "}.lst-kix_6ftjd98kskfz-8>li:before{content:"\0025a0   "}.lst-kix_9rbk2sol80ch-2>li:before{content:"\0025a0   "}.lst-kix_9rbk2sol80ch-3>li:before{content:"\0025cf   "}.lst-kix_ibkhg9oor910-8>li:before{content:"\0025a0   "}.lst-kix_ibkhg9oor910-7>li:before{content:"\0025cb   "}.lst-kix_9rbk2sol80ch-0>li:before{content:"\0025cf   "}.lst-kix_9rbk2sol80ch-1>li:before{content:"\0025cb   "}.lst-kix_9rbk2sol80ch-4>li:before{content:"\0025cb   "}.lst-kix_9rbk2sol80ch-5>li:before{content:"\0025a0   "}ul.lst-kix_6ftjd98kskfz-8{list-style-type:none}ul.lst-kix_6ftjd98kskfz-7{list-style-type:none}.lst-kix_9rbk2sol80ch-6>li:before{content:"\0025cf   "}.lst-kix_9rbk2sol80ch-7>li:before{content:"\0025cb   "}ul.lst-kix_d1ebcswm62mz-3{list-style-type:none}ul.lst-kix_d1ebcswm62mz-2{list-style-type:none}ul.lst-kix_d1ebcswm62mz-1{list-style-type:none}ul.lst-kix_d1ebcswm62mz-0{list-style-type:none}ul.lst-kix_9b6sqx4ewyqe-0{list-style-type:none}ul.lst-kix_d1ebcswm62mz-7{list-style-type:none}ul.lst-kix_9b6sqx4ewyqe-1{list-style-type:none}ul.lst-kix_d1ebcswm62mz-6{list-style-type:none}ul.lst-kix_9b6sqx4ewyqe-2{list-style-type:none}ul.lst-kix_d1ebcswm62mz-5{list-style-type:none}ul.lst-kix_9b6sqx4ewyqe-3{list-style-type:none}ul.lst-kix_d1ebcswm62mz-4{list-style-type:none}ul.lst-kix_9b6sqx4ewyqe-4{list-style-type:none}ul.lst-kix_9b6sqx4ewyqe-5{list-style-type:none}ul.lst-kix_9b6sqx4ewyqe-6{list-style-type:none}ul.lst-kix_9b6sqx4ewyqe-7{list-style-type:none}ul.lst-kix_9b6sqx4ewyqe-8{list-style-type:none}.lst-kix_tg2n18lwzkp6-7>li:before{content:"\0025cb   "}ul.lst-kix_6ftjd98kskfz-0{list-style-type:none}ul.lst-kix_6ftjd98kskfz-2{list-style-type:none}ul.lst-kix_6ftjd98kskfz-1{list-style-type:none}.lst-kix_tg2n18lwzkp6-8>li:before{content:"\0025a0   "}ul.lst-kix_6ftjd98kskfz-4{list-style-type:none}ul.lst-kix_6ftjd98kskfz-3{list-style-type:none}.lst-kix_9rbk2sol80ch-8>li:before{content:"\0025a0   "}ul.lst-kix_6ftjd98kskfz-6{list-style-type:none}ul.lst-kix_6ftjd98kskfz-5{list-style-type:none}ul.lst-kix_jxivkebb27r4-2{list-style-type:none}ul.lst-kix_jxivkebb27r4-1{list-style-type:none}ul.lst-kix_jxivkebb27r4-4{list-style-type:none}ul.lst-kix_jxivkebb27r4-3{list-style-type:none}ul.lst-kix_jxivkebb27r4-0{list-style-type:none}.lst-kix_6ftjd98kskfz-4>li:before{content:"\0025cb   "}ul.lst-kix_9rbk2sol80ch-2{list-style-type:none}ul.lst-kix_9rbk2sol80ch-3{list-style-type:none}ul.lst-kix_9rbk2sol80ch-0{list-style-type:none}ul.lst-kix_9rbk2sol80ch-1{list-style-type:none}.lst-kix_6ftjd98kskfz-2>li:before{content:"\0025a0   "}.lst-kix_6ftjd98kskfz-3>li:before{content:"\0025cf   "}.lst-kix_6ftjd98kskfz-0>li:before{content:"\0025cf   "}.lst-kix_6ftjd98kskfz-1>li:before{content:"\0025cb   "}ul.lst-kix_9rbk2sol80ch-8{list-style-type:none}ul.lst-kix_9rbk2sol80ch-6{list-style-type:none}ul.lst-kix_9rbk2sol80ch-7{list-style-type:none}ul.lst-kix_9rbk2sol80ch-4{list-style-type:none}ul.lst-kix_9rbk2sol80ch-5{list-style-type:none}.lst-kix_9b6sqx4ewyqe-1>li:before{content:"\0025cb   "}ul.lst-kix_jxivkebb27r4-6{list-style-type:none}ul.lst-kix_jxivkebb27r4-5{list-style-type:none}ul.lst-kix_jxivkebb27r4-8{list-style-type:none}.lst-kix_9b6sqx4ewyqe-0>li:before{content:"\0025cf   "}ul.lst-kix_jxivkebb27r4-7{list-style-type:none}.lst-kix_9b6sqx4ewyqe-3>li:before{content:"\0025cf   "}.lst-kix_9b6sqx4ewyqe-5>li:before{content:"\0025a0   "}.lst-kix_tg2n18lwzkp6-0>li:before{content:"\0025cf   "}.lst-kix_9b6sqx4ewyqe-2>li:before{content:"\0025a0   "}.lst-kix_9b6sqx4ewyqe-6>li:before{content:"\0025cf   "}.lst-kix_tg2n18lwzkp6-1>li:before{content:"\0025cb   "}ul.lst-kix_jzj2m2rfzlp6-8{list-style-type:none}.lst-kix_9b6sqx4ewyqe-4>li:before{content:"\0025cb   "}ul.lst-kix_jzj2m2rfzlp6-1{list-style-type:none}ul.lst-kix_jzj2m2rfzlp6-0{list-style-type:none}ul.lst-kix_jzj2m2rfzlp6-3{list-style-type:none}ul.lst-kix_jzj2m2rfzlp6-2{list-style-type:none}.lst-kix_tg2n18lwzkp6-6>li:before{content:"\0025cf   "}ul.lst-kix_jzj2m2rfzlp6-5{list-style-type:none}ul.lst-kix_jzj2m2rfzlp6-4{list-style-type:none}ul.lst-kix_jzj2m2rfzlp6-7{list-style-type:none}ul.lst-kix_jzj2m2rfzlp6-6{list-style-type:none}ul.lst-kix_tg2n18lwzkp6-0{list-style-type:none}.lst-kix_tg2n18lwzkp6-5>li:before{content:"\0025a0   "}ul.lst-kix_tg2n18lwzkp6-1{list-style-type:none}ul.lst-kix_tg2n18lwzkp6-2{list-style-type:none}ul.lst-kix_tg2n18lwzkp6-3{list-style-type:none}.lst-kix_9b6sqx4ewyqe-7>li:before{content:"\0025cb   "}.lst-kix_tg2n18lwzkp6-2>li:before{content:"\0025a0   "}.lst-kix_tg2n18lwzkp6-4>li:before{content:"\0025cb   "}ul.lst-kix_tg2n18lwzkp6-4{list-style-type:none}ul.lst-kix_tg2n18lwzkp6-5{list-style-type:none}ul.lst-kix_tg2n18lwzkp6-6{list-style-type:none}ul.lst-kix_tg2n18lwzkp6-7{list-style-type:none}.lst-kix_9b6sqx4ewyqe-8>li:before{content:"\0025a0   "}.lst-kix_tg2n18lwzkp6-3>li:before{content:"\0025cf   "}ul.lst-kix_tg2n18lwzkp6-8{list-style-type:none}.lst-kix_jxivkebb27r4-0>li:before{content:"\0025cf   "}.lst-kix_jxivkebb27r4-1>li:before{content:"\0025cb   "}.lst-kix_d1ebcswm62mz-0>li:before{content:"\0025cf   "}ul.lst-kix_ibkhg9oor910-2{list-style-type:none}ul.lst-kix_ibkhg9oor910-1{list-style-type:none}ul.lst-kix_ibkhg9oor910-0{list-style-type:none}.lst-kix_jxivkebb27r4-4>li:before{content:"\0025cb   "}ul.lst-kix_d1ebcswm62mz-8{list-style-type:none}.lst-kix_jxivkebb27r4-5>li:before{content:"\0025a0   "}.lst-kix_jxivkebb27r4-6>li:before{content:"\0025cf   "}.lst-kix_d1ebcswm62mz-6>li:before{content:"\0025cf   "}.lst-kix_d1ebcswm62mz-5>li:before{content:"\0025a0   "}.lst-kix_ibkhg9oor910-2>li:before{content:"\0025a0   "}.lst-kix_d1ebcswm62mz-4>li:before{content:"\0025cb   "}.lst-kix_ibkhg9oor910-3>li:before{content:"\0025cf   "}.lst-kix_jzj2m2rfzlp6-7>li:before{content:"\0025cb   "}.lst-kix_d1ebcswm62mz-2>li:before{content:"\0025a0   "}.lst-kix_ibkhg9oor910-5>li:before{content:"\0025a0   "}ul.lst-kix_ibkhg9oor910-6{list-style-type:none}.lst-kix_jzj2m2rfzlp6-8>li:before{content:"\0025a0   "}ul.lst-kix_ibkhg9oor910-5{list-style-type:none}.lst-kix_jxivkebb27r4-3>li:before{content:"\0025cf   "}.lst-kix_d1ebcswm62mz-1>li:before{content:"\0025cb   "}.lst-kix_d1ebcswm62mz-3>li:before{content:"\0025cf   "}.lst-kix_ibkhg9oor910-4>li:before{content:"\0025cb   "}ul.lst-kix_ibkhg9oor910-4{list-style-type:none}.lst-kix_ibkhg9oor910-6>li:before{content:"\0025cf   "}ul.lst-kix_ibkhg9oor910-3{list-style-type:none}.lst-kix_jxivkebb27r4-2>li:before{content:"\0025a0   "}ul.lst-kix_ibkhg9oor910-8{list-style-type:none}ul.lst-kix_ibkhg9oor910-7{list-style-type:none}.lst-kix_jzj2m2rfzlp6-6>li:before{content:"\0025cf   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_jzj2m2rfzlp6-5>li:before{content:"\0025a0   "}.lst-kix_ibkhg9oor910-1>li:before{content:"\0025cb   "}.lst-kix_jzj2m2rfzlp6-4>li:before{content:"\0025cb   "}.lst-kix_jxivkebb27r4-7>li:before{content:"\0025cb   "}.lst-kix_jxivkebb27r4-8>li:before{content:"\0025a0   "}.lst-kix_d1ebcswm62mz-7>li:before{content:"\0025cb   "}.lst-kix_ibkhg9oor910-0>li:before{content:"\0025cf   "}.lst-kix_d1ebcswm62mz-8>li:before{content:"\0025a0   "}.lst-kix_jzj2m2rfzlp6-3>li:before{content:"\0025cf   "}.lst-kix_jzj2m2rfzlp6-0>li:before{content:"\0025cf   "}.lst-kix_jzj2m2rfzlp6-2>li:before{content:"\0025a0   "}.lst-kix_jzj2m2rfzlp6-1>li:before{content:"\0025cb   "}ol{margin:0;padding:0}table td,table th{padding:0}.c20{margin-left:72pt;padding-top:0pt;padding-left:0pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c0{margin-left:36pt;padding-top:0pt;padding-left:0pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c10{margin-left:36pt;padding-top:12pt;padding-left:0pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c15{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left;margin-right:4pt}.c9{padding-top:12pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left;height:11pt}.c13{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c12{padding-top:12pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c23{padding-top:8pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c16{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c3{text-decoration-skip-ink:none;font-size:12pt;-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline}.c5{font-weight:400;text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}.c7{font-weight:400;vertical-align:baseline;font-family:"Arial";font-style:normal}.c8{text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}.c1{font-size:12pt;font-style:italic;color:#111111}.c19{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c11{-webkit-text-decoration-skip:none;text-decoration:underline;text-decoration-skip-ink:none}.c2{color:#111111;font-size:12pt}.c14{padding:0;margin:0}.c18{color:#555555;font-size:12pt}.c17{color:#111111;font-size:23pt}.c24{color:#000000;font-size:11pt}.c4{color:inherit;text-decoration:inherit}.c22{height:11pt}.c21{font-style:italic}.c6{font-weight:700}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c19 doc-content"><p class="c16"><span class="c8 c17 c6">Jie Ren</span></p><h1 class="c16" id="h.nh88b2avfnl1"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 120.00px; height: 120.00px;"><img alt="" src="images/profile_photo.jpg" style="width: 120.00px; height: 120.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h1><p class="c16"><span class="c5 c2">Computational Biology and Bioinformatics; Machine learning; Statistical Modeling</span></p><p class="c15"><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://twitter.com/jessierenjie&amp;sa=D&amp;source=editors&amp;ust=1702058950706347&amp;usg=AOvVaw224TvWiZyZOKQOlwW8XRql">Twitter</a></span><span class="c2">&nbsp;</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/jessieren&amp;sa=D&amp;source=editors&amp;ust=1702058950706483&amp;usg=AOvVaw3x9UDrBji5-EjgKMvpBnxT">Github</a></span><span class="c2">&nbsp;</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://scholar.google.com/citations?user%3DOs9wmpkAAAAJ%26hl%3Den&amp;sa=D&amp;source=editors&amp;ust=1702058950706595&amp;usg=AOvVaw0W_IE9thFTFm8QHFALayR1">Google Scholar</a></span><span class="c2">&nbsp;</span></p><p class="c15"><span class="c2">E-mail: jie dot ren dot usc (at) gmail.com</span></p><p class="c23"><span class="c5 c18">&nbsp;</span></p><p class="c16"><span class="c2">I am a Senior Research Scientist in </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://research.google/teams/brain/&amp;sa=D&amp;source=editors&amp;ust=1702058950706898&amp;usg=AOvVaw2klNBdsjbHFjWcndQUAzeh">Google Research Brain Team</a></span><span class="c2">. Previously, I received my PhD in </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.qcb-dornsife.usc.edu/&amp;sa=D&amp;source=editors&amp;ust=1702058950706992&amp;usg=AOvVaw23SDPGgycNi8SlIagz_q_e">Computational Biology and Bioinformatics</a></span><span class="c5 c2">&nbsp;and MSc in Statistics at the University of Southern California. I am interested in three primary areas, (1) out-of-distribution (OOD) detection in deep learning, (2) uncertainty estimation and robustness of large language/ vision models, and (3) the development of accurate and reliable machine learning algorithms and statistical methods for biological and medical studies. My objective is to develop reliable AI solutions that can be safely deployed in real-world applications, advancing progress in sciences and humanity.</span></p><p class="c13"><span class="c8 c2 c6">News</span></p><ul class="c14 lst-kix_jzj2m2rfzlp6-0 start"><li class="c10 li-bullet-0"><span class="c2">12/2023: Our study on selective generation for large language models, &quot;Self-evaluation improves selective generation in large language models&rdquo;, will be presented (spotlight) at NeurIPS 2023 I Can&rsquo;t Believe It&rsquo;s Not Better! (ICBINB) Workshop [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://openreview.net/forum?id%3DOptKBWmreP&amp;sa=D&amp;source=editors&amp;ust=1702058950707252&amp;usg=AOvVaw1n8LgzsGGPxs8E6IFs5DeD">paper</a></span><span class="c5 c2">].</span></li><li class="c0 li-bullet-0"><span class="c2">12/2023: Two papers studying the uncertainty estimation in language generation models are accepted by ENNLP 2023: &ldquo;Improving the Robustness of Summarization Models by Detecting and Removing Input Noise&rdquo;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2212.09928&amp;sa=D&amp;source=editors&amp;ust=1702058950707406&amp;usg=AOvVaw0eGTlGqqgWiizC0k3HbKSu">paper]</a></span><span class="c2">, and &ldquo;On Uncertainty Calibration and Selective Generation in Probabilistic Neural Summarization: A Benchmark Study</span><span class="c2 c6">&quot;</span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2304.08653&amp;sa=D&amp;source=editors&amp;ust=1702058950707533&amp;usg=AOvVaw15Xgte-PQst5Up6HS_i4Ak">paper</a></span><span class="c5 c2">].</span></li><li class="c0 li-bullet-0"><span class="c2">07/2023: Our work on prompt selection and prompt ensemble for multimodal models, &quot;A simple zero-shot prompt weighting technique to improve prompt ensembling in text-image models&quot;, will be presented at ICML 2023. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://proceedings.mlr.press/v202/allingham23a.html&amp;sa=D&amp;source=editors&amp;ust=1702058950707700&amp;usg=AOvVaw1veyhwjFYjwBNZOsNdhLzw">paper]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://twitter.com/JamesAllingham/status/1684125147756888064&amp;sa=D&amp;source=editors&amp;ust=1702058950707798&amp;usg=AOvVaw1nizY7YSmFqsY36CS7IlRF">twitter]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/google/uncertainty-baselines/tree/main/experimental/multimodal&amp;sa=D&amp;source=editors&amp;ust=1702058950707905&amp;usg=AOvVaw294hQePjvz9sEWmJrUC4_Z">code]</a></span><span class="c5 c2">&nbsp;.</span></li><li class="c0 li-bullet-0"><span class="c2">06/2023: Our paper &quot;Improving Zero-shot Generalization and Robustness of Multi-modal Models&quot; is accepted by CVPR 2023. Congratulations to Yunhao Ge. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/pdf/2212.01758.pdf&amp;sa=D&amp;source=editors&amp;ust=1702058950708052&amp;usg=AOvVaw19noptQ56aGhEMQF54vJb8">paper]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://twitter.com/GeYunhao/status/1664478907012366337&amp;sa=D&amp;source=editors&amp;ust=1702058950708158&amp;usg=AOvVaw223ZANLsAgT_6xXu_kVD0S">twitter]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/gyhandy/Hierarchy-CLIP&amp;sa=D&amp;source=editors&amp;ust=1702058950708246&amp;usg=AOvVaw3FBlXxDLLScekDsLyHu7MA">code]</a></span><span class="c5 c2">&nbsp;.</span></li><li class="c0 li-bullet-0"><span class="c2">05/2023: Our paper &quot;Out-of-Distribution Detection and Selective Generation for Conditional Language Models&quot; is accepted (spotlight) by ICLR 2023. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2209.15558&amp;sa=D&amp;source=editors&amp;ust=1702058950708389&amp;usg=AOvVaw1YpJdH7ZGxKmNzVcNwvX1x">paper]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://twitter.com/peterjliu/status/1620157580114030592&amp;sa=D&amp;source=editors&amp;ust=1702058950708480&amp;usg=AOvVaw0pGQiD-DnleYswVdg27Usn">twitter]</a></span><span class="c5 c2">&nbsp;.</span></li><li class="c0 li-bullet-0"><span class="c2">07/2022: Together with my colleagues at Google, we release a framework, Plex, for reliability in large pre-trained models. Please check out our latest blog post to learn more. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://ai.googleblog.com/2022/07/towards-reliability-in-deep-learning.html&amp;sa=D&amp;source=editors&amp;ust=1702058950708684&amp;usg=AOvVaw06u-_cgijOZDcQWofr1QRT">blog]</a></span><span class="c5 c2">&nbsp;.</span></li><li class="c0 li-bullet-0"><span class="c2">01/2022: Our new AI blog post is out :) It walks through our recent paper on enabling dermatology AI systems to detect the presence of conditinos unseen in training [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://ai.googleblog.com/2022/01/does-your-medical-image-classifier-know.html&amp;sa=D&amp;source=editors&amp;ust=1702058950708858&amp;usg=AOvVaw3JikYs6W6oja2RT7Vj4GAC">blog]</a></span><span class="c5 c2">&nbsp;.</span></li><li class="c0 li-bullet-0"><span class="c2">12/2021: Our paper &quot;Exploring the Limits of Out-of-Distribution Detection&quot; [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2106.03004&amp;sa=D&amp;source=editors&amp;ust=1702058950708990&amp;usg=AOvVaw1PDVHdx5595HNz2uUI3Owk">paper]</a></span><span class="c5 c2">&nbsp;will be presented at NeurIPS 2021.</span></li><li class="c0 li-bullet-0"><span class="c2">07/2021: Together with </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://polkirichenko.github.io/&amp;sa=D&amp;source=editors&amp;ust=1702058950709129&amp;usg=AOvVaw0mzA0kpBI4L0CDHJRH2_4B">Polina Kirichenko</a></span><span class="c2">, </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://scholar.google.com/citations?user%3D-SjNMu8AAAAJ%26hl%3Den&amp;sa=D&amp;source=editors&amp;ust=1702058950709239&amp;usg=AOvVaw0AEnTMzn2f50ApXeDHcoqK">Haleh Akrami</a></span><span class="c2">, </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=http://pages.cs.wisc.edu/~sharonli/&amp;sa=D&amp;source=editors&amp;ust=1702058950709324&amp;usg=AOvVaw3RaAM6PxsUq3vWvhmw4waC">Sharon Yixuan Li</a></span><span class="c2">&nbsp;and </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://scholar.google.com/citations?user%3DD_CuHyUAAAAJ%26hl%3Den&amp;sa=D&amp;source=editors&amp;ust=1702058950709426&amp;usg=AOvVaw1GASn2ckQjHwIQZ9-icig8">Serg&uuml;l Ayd&ouml;re</a></span><span class="c2">, we are going to lead a </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://icml.cc/virtual/2021/affinityworkshop/10849&amp;sa=D&amp;source=editors&amp;ust=1702058950709520&amp;usg=AOvVaw0Snd01AcAlBWxtMbCaqoII">breakout session</a></span><span class="c5 c2">&nbsp;at July 21 7:25 pm ET at WiML Un-Workshop at ICML 2021 on &quot;Does your model know what it doesn&rsquo;t know? Uncertainty estimation and OOD detection in deep learning.&quot;</span></li><li class="c0 li-bullet-0"><span class="c2">07/2021: Our paper &quot;A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection&quot; [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2106.09022&amp;sa=D&amp;source=editors&amp;ust=1702058950709684&amp;usg=AOvVaw1pwrrg9fEF9w5hyMfhC8DU">paper]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://twitter.com/jessierenjie/status/1418321220546613248&amp;sa=D&amp;source=editors&amp;ust=1702058950709779&amp;usg=AOvVaw0h5zn0fYedQVEsKPxAw8lP">poster]</a></span><span class="c5 c2">&nbsp;will be presented at ICML 2021 UDL workshop.</span></li><li class="c0 li-bullet-0"><span class="c2">07/2021: Our paper &quot;Exploring the Limits of Out-of-Distribution Detection&quot; [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2106.03004&amp;sa=D&amp;source=editors&amp;ust=1702058950709918&amp;usg=AOvVaw3dqj-T4CXnJViWV91kuLKc">paper]</a></span><span class="c5 c2">&nbsp;will be presented at ICML 2021 UDL workshop.</span></li><li class="c0 li-bullet-0"><span class="c2">12/2019: We will present our work of &quot;Likelihood Ratios for Out-of-Distribution Detection&quot; at NeurIPS 2019. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/1906.02845&amp;sa=D&amp;source=editors&amp;ust=1702058950710061&amp;usg=AOvVaw2uF77hvLveFaR8HQ6ZnENx">paper</a></span><span class="c2">][</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/1906.02845&amp;sa=D&amp;source=editors&amp;ust=1702058950710141&amp;usg=AOvVaw1I30TeryFJ5Hwlk1ymBJ0h">paper] [</a></span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/google-research/google-research/tree/master/genomics_ood&amp;sa=D&amp;source=editors&amp;ust=1702058950710225&amp;usg=AOvVaw375fkM22fEoemMVOCfTXkV">code] [</a></span><span class="c3"><a class="c4" href="https://www.google.com/url?q=http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-llr-ood-poster.pdf&amp;sa=D&amp;source=editors&amp;ust=1702058950710310&amp;usg=AOvVaw1Pv2gj2HmQZT_t-jcYxKCh">poster] [</a></span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.youtube.com/watch?v%3D-FduW9ZWAR4&amp;sa=D&amp;source=editors&amp;ust=1702058950710376&amp;usg=AOvVaw1_Myd29wRYP9VNaxt-9B44">3-minute video] </a></span><span class="c2">[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://ai.googleblog.com/2019/12/improving-out-of-distribution-detection.html&amp;sa=D&amp;source=editors&amp;ust=1702058950710491&amp;usg=AOvVaw0FOzo55AX6Bua7bmHiawGh">blog]</a></span></li><li class="c0 li-bullet-0"><span class="c5 c2">06/2019: I am joining Google Brain as a Research Scientist. </span></li><li class="c0 li-bullet-0"><span class="c5 c2">06/2018: I am joining Google Brain AI Residency program. </span></li><li class="c0 li-bullet-0"><span class="c2">03/2018: Our work is highlighted in </span><span class="c3 c21"><a class="c4" href="https://www.google.com/url?q=https://www.nature.com/articles/d41586-018-03358-3&amp;sa=D&amp;source=editors&amp;ust=1702058950710750&amp;usg=AOvVaw3OuPFDFBbRRFXVPghWUD1y">Nature News </a></span><span class="c5 c2">titled &quot;Machine learning spots treasure trove of elusive viruses Artificial intelligence could speed up metagenomic studies that look for species unknown to science.&quot;.</span></li><li class="c0 li-bullet-0"><span class="c2">04/2017: I am honored to receive the prestigious </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://twitter.com/uscbiosci/status/858070439435710464&amp;sa=D&amp;source=editors&amp;ust=1702058950710913&amp;usg=AOvVaw2HmV_NOyvnwvzwZeVmWYjL">USC PhD Achievement Award</a></span><span class="c5 c2">&nbsp;(6 winners university-wide per year).</span></li><li class="c0 li-bullet-0"><span class="c5 c2">03/2017: I am honored to receive Women in Science and Engineering Travel Grant. </span></li><li class="c10 li-bullet-0"><span class="c2">04/2016: I am honored to receive </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://dornsife.usc.edu/cams/cams-prize/&amp;sa=D&amp;source=editors&amp;ust=1702058950711108&amp;usg=AOvVaw39MLcnhVzmNOrrvEc005DQ">USC CAMS Graduate Student Prize</a></span><span class="c5 c2">&nbsp;for Excellence in Research with a Substantial Mathematical Component (2 winners university-wide per year). </span></li></ul><p class="c12"><span class="c5 c2">&nbsp;</span></p><p class="c12"><span class="c8 c2 c6">Publications</span></p><p class="c12"><span class="c8 c2 c6">Out-of-Distribution Detection in Deep Learning</span></p><ul class="c14 lst-kix_jxivkebb27r4-0 start"><li class="c10 li-bullet-0"><span class="c2 c6">Building One-class Detector for Anything: Open-vocabulary Zero-shot OOD Detection Using Text-image Models. </span><span class="c2">Presented at the </span><span class="c1">ICML workshop on Knowledge and Logical Reasoning in the Era of Data-driven Learning &nbsp;</span><span class="c2">(2023). &nbsp;Yunhao Ge*, </span><span class="c1">Jie Ren*</span><span class="c2">, Jiaping Zhao, Kaifeng Chen, Andrew Gallagher, Laurent Itti, Balaji Lakshminarayanan.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2305.17207&amp;sa=D&amp;source=editors&amp;ust=1702058950711498&amp;usg=AOvVaw1ssoFWU7QDMHDD6rf-_nGu">paper]</a></span><span class="c5 c2">&nbsp; &nbsp;</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Out-of-Distribution Detection and Selective Generation for Conditional Language Models. </span><span class="c1">ICLR</span><span class="c2">&nbsp;(2023). &nbsp;</span><span class="c1">Jie Ren</span><span class="c2">, Jiaming Luo, Yao Zhao, Kundan Krishna, Mohammad Saleh, Balaji Lakshminarayanan, Peter J. Liu.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2209.15558&amp;sa=D&amp;source=editors&amp;ust=1702058950711746&amp;usg=AOvVaw15n3oDphKJ2MDsL24w5D-Z">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">A Simple Approach to Improve Single-Model Deep Uncertainty via Distance-Awareness. </span><span class="c1">Journal of Machine Learning Research</span><span class="c2">&nbsp;23 (2022): 1-63. &nbsp;Jeremiah Zhe Liu*, Shreyas Padhy*, </span><span class="c1">Jie Ren*</span><span class="c2">, Zi Lin, Yeming Wen, Ghassen Jerfel, Zack Nado, Jasper Snoek, Dustin Tran, and Balaji Lakshminarayanan.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.jmlr.org/papers/volume24/22-0479/22-0479.pdf&amp;sa=D&amp;source=editors&amp;ust=1702058950712057&amp;usg=AOvVaw1f53ssuD9NFQfcZ7o24IgU">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">MLR-OOD: A Markov Chain Based Likelihood Ratio Method for Out-Of-Distribution Detection of Genomic Sequences. </span><span class="c1">Journal of Molecular Biology</span><span class="c2">&nbsp;(2022). &nbsp;Xin Bai, </span><span class="c1">Jie Ren</span><span class="c2">, Fengzhu Sun.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.sciencedirect.com/science/article/pii/S0022283622001668&amp;sa=D&amp;source=editors&amp;ust=1702058950712326&amp;usg=AOvVaw1YFukqq-jxelMzrdGcWA5Y">paper]</a></span><span class="c2">[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://docs.google.com/presentation/d/1XmsWPViWmXh0cwOBFRXlBE2CnsWyaynzN9Qrmv2pNRA/edit?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1702058950712450&amp;usg=AOvVaw3vdWXfnc44C-0fVgio0s4A">slides]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">Exploring the Limits of Out-of-Distribution Detection. </span><span class="c1">NeurIPS</span><span class="c2">&nbsp;(2021). &nbsp;Stanislav Fort*, </span><span class="c1">Jie Ren*</span><span class="c2">, and Balaji Lakshminarayanan.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2106.03004&amp;sa=D&amp;source=editors&amp;ust=1702058950712674&amp;usg=AOvVaw1K_fqeG5SEcIvK-ovqih0U">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection. &nbsp;</span><span class="c2">Presented at the </span><span class="c1">ICML workshop on Uncertainty and Robustness in Deep Learning</span><span class="c2">&nbsp;(2021). &nbsp;</span><span class="c1">Jie Ren</span><span class="c2">, Stanislav Fort, Jeremiah Liu, Abhijit Guha Roy, Shreyas Padhy, and Balaji Lakshminarayanan. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2106.09022&amp;sa=D&amp;source=editors&amp;ust=1702058950712916&amp;usg=AOvVaw3Im2AyyWqzQsvtW_RnOpRU">paper] </a></span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://twitter.com/jessierenjie/status/1418321220546613248&amp;sa=D&amp;source=editors&amp;ust=1702058950712994&amp;usg=AOvVaw1L70DICncLTU3bY-DjFYeI">[poster]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/google/uncertainty-baselines/blob/main/experimental/near_ood/vit/ood_utils.py&amp;sa=D&amp;source=editors&amp;ust=1702058950713111&amp;usg=AOvVaw188N6RDuMpO9OaIdXnbbmQ">code</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Does Your Dermatology Classifier Know What It Doesn&#39;t Know? Detecting the Long-Tail of Unseen Conditions.</span><span class="c2">&nbsp;</span><span class="c1">Medical Image Analysis</span><span class="c2">&nbsp;75 (2022): 102274. Abhijit Guha Roy*, </span><span class="c1">Jie Ren*</span><span class="c2">, Shekoofeh Azizi, Aaron Loh, Vivek Natarajan, Basil Mustafa, Nick Pawlowski et al. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2104.03829&amp;sa=D&amp;source=editors&amp;ust=1702058950713362&amp;usg=AOvVaw2S92LAIpdVkOd2H51DBKUZ">paper]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://ai.googleblog.com/2022/01/does-your-medical-image-classifier-know.html&amp;sa=D&amp;source=editors&amp;ust=1702058950713472&amp;usg=AOvVaw3GaG4NlpxQyutZiGdoam_2">blog</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Revisiting One-vs-All Classifiers for Predictive Uncertainty and Out-of-Distribution Detection in Neural Networks. &nbsp;</span><span class="c2">Presented at the </span><span class="c1">ICML workshop on Uncertainty and Robustness in Deep Learning</span><span class="c2">&nbsp;(2020). &nbsp;Shreyas Padhy, Zachary Nado, </span><span class="c1">Jie Ren,</span><span class="c2">&nbsp;Jeremiah Liu, Jasper Snoek, and Balaji Lakshminarayanan. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2007.05134&amp;sa=D&amp;source=editors&amp;ust=1702058950713736&amp;usg=AOvVaw3FbU2A2rDCs7IBz3KG7XUN">paper]</a></span></li><li class="c10 li-bullet-0"><span class="c2 c6">Likelihood Ratios for Out-of-Distribution Detection. </span><span class="c1">NeurIPS</span><span class="c2">&nbsp;(2019). </span><span class="c1">Jie Ren</span><span class="c2">, Peter J. Liu, Emily Fertig, Jasper Snoek, Ryan Poplin, Mark A. DePristo, Joshua V. Dillon, and Balaji Lakshminarayanan. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/1906.02845&amp;sa=D&amp;source=editors&amp;ust=1702058950713958&amp;usg=AOvVaw0rYfcZwi6An0TQMqpWgGq8">paper] [</a></span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/google-research/google-research/tree/master/genomics_ood&amp;sa=D&amp;source=editors&amp;ust=1702058950714044&amp;usg=AOvVaw0yO1OqBdB1mBuJIZZcCVUY">code] [</a></span><span class="c3"><a class="c4" href="https://www.google.com/url?q=http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-llr-ood-poster.pdf&amp;sa=D&amp;source=editors&amp;ust=1702058950714131&amp;usg=AOvVaw1qxwJ-KtdRhMSgKOgccrCH">poster] [</a></span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.youtube.com/watch?v%3D-FduW9ZWAR4&amp;sa=D&amp;source=editors&amp;ust=1702058950714201&amp;usg=AOvVaw3lPUthjuTMVrtuTFpyIdDO">3-minute video] </a></span><span class="c2">[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://ai.googleblog.com/2019/12/improving-out-of-distribution-detection.html&amp;sa=D&amp;source=editors&amp;ust=1702058950714308&amp;usg=AOvVaw30GMlmk6eJQh_zJB19uit4">blog]</a></span><span class="c5 c2">&nbsp; </span></li></ul><p class="c9"><span class="c8 c2 c6"></span></p><p class="c12"><span class="c8 c2 c6">Uncertainty and Robustness in Deep Learning</span></p><ul class="c14 lst-kix_9b6sqx4ewyqe-0 start"><li class="c10 li-bullet-0"><span class="c2 c6">Universal Self-Consistency for Large Language Model Generation.</span><span class="c2">&nbsp;arXiv preprint arXiv:2311.17311 (2023). Chen, Xinyun, Renat Aksitov, Uri Alon, </span><span class="c1">Jie Ren</span><span class="c5 c2">, Kefan Xiao, Pengcheng Yin, Sushant Prakash, Charles Sutton, Xuezhi Wang, and Denny Zhou. </span></li><li class="c0 li-bullet-0"><span class="c2 c6">On Uncertainty Calibration and Selective Generation in Probabilistic Neural Summarization: A Benchmark Study. </span><span class="c1">EMNLP</span><span class="c2">&nbsp;(2023). Polina Zablotskaia, Du Phan, Joshua Maynez, Shashi Narayan, </span><span class="c1">Jie Ren</span><span class="c2">, Jeremiah Liu. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2304.08653&amp;sa=D&amp;source=editors&amp;ust=1702058950714798&amp;usg=AOvVaw1_EDQUl2My2BTBlIpV82aC">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Improving the Robustness of Summarization Models by Detecting and Removing Input Noise. </span><span class="c1">EMNLP</span><span class="c2">&nbsp;(2023). &nbsp;Kundan Krishna, Yao Zhao, </span><span class="c1">Jie Ren</span><span class="c2">, Balaji Lakshminarayanan, Jiaming Luo, Mohammad Saleh, Peter J. Liu.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2212.09928&amp;sa=D&amp;source=editors&amp;ust=1702058950715087&amp;usg=AOvVaw0gLdoRVJV2ghvfEmi30itE">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">A Simple Zero-shot Prompt Weighting Technique to Improve Prompt Ensembling in Text-Image Models. </span><span class="c1">ICML</span><span class="c2">&nbsp;(2023). &nbsp;James Urquhart Allingham*, </span><span class="c1">Jie Ren*</span><span class="c2">, Michael W Dusenberry, Jeremiah Zhe Liu, Xiuye Gu, Yin Cui, Dustin Tran, Balaji Lakshminarayanan.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2302.06235&amp;sa=D&amp;source=editors&amp;ust=1702058950715331&amp;usg=AOvVaw1PBonaP8jGp1plKGiz5y1v">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">Morse Neural Networks for Uncertainty Quantification.</span><span class="c2">&nbsp;Presented at</span><span class="c1">&nbsp;the ICML workshop on</span><span class="c2">&nbsp;</span><span class="c1">Structured Probabilistic Inference and Generative Modeling </span><span class="c2">Benoit Dherin, Huiyi Hu, </span><span class="c1">Jie Ren</span><span class="c2">, Michael W. Dusenberry, and Balaji Lakshminarayanan.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2307.00667&amp;sa=D&amp;source=editors&amp;ust=1702058950715651&amp;usg=AOvVaw2TViplx2yX35MHUydd9HNJ">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Improving Zero-shot Generalization and Robustness of Multi-modal Models. </span><span class="c1">CVPR</span><span class="c2">&nbsp;(2022). &nbsp;Yunhao Ge*, </span><span class="c1">Jie Ren*</span><span class="c2">, Yuxiao Wang, Andrew Gallagher, Ming-Hsuan Yang, Laurent Itti, Hartwig Adam, Balaji Lakshminarayanan, Jiaping Zhao.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2212.01758&amp;sa=D&amp;source=editors&amp;ust=1702058950715878&amp;usg=AOvVaw1AnZB43NgLZi-efYl0GKG1">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">Plex: Towards reliability using pretrained large model extensions. </span><span class="c1">arXiv preprint arXiv:2207.07411</span><span class="c2">&nbsp;(2022). &nbsp;Dustin Tran, Jeremiah Liu, Michael W Dusenberry, Du Phan, Mark Collier, </span><span class="c1">Jie Ren</span><span class="c2">, Kehang Han, Zi Wang, Zelda Mariet, Huiyi Hu, Neil Band, Tim GJ Rudner, Karan Singhal, Zachary Nado, Joost van Amersfoort, Andreas Kirsch, Rodolphe Jenatton, Nithum Thain, Honglin Yuan, Kelly Buchanan, Kevin Murphy, D Sculley, Yarin Gal, Zoubin Ghahramani, Jasper Snoek, Balaji Lakshminarayanan.[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2207.07411&amp;sa=D&amp;source=editors&amp;ust=1702058950716135&amp;usg=AOvVaw3qcVKtmo_YUCr76XRr4Uoa">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">Uncertainty Baselines: Benchmarks for Uncertainty &amp; Robustness in Deep Learning. </span><span class="c2">Presented at the </span><span class="c1">NeurIPS workshop on Bayesian deep learning</span><span class="c2">&nbsp;(2021). Zachary Nado, Neil Band, Mark Collier, Josip Djolonga, Michael W. Dusenberry, Sebastian Farquhar, Angelos Filos et al. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/2106.04015&amp;sa=D&amp;source=editors&amp;ust=1702058950716345&amp;usg=AOvVaw0R1lxV7DwiZzek8XkwYN04">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">SummAE: Summae: Zero-Shot Abstractive Text Summarization Using Length-Agnostic Auto-Encoders.</span><span class="c2">&nbsp;</span><span class="c1">arXiv preprint arXiv:1910.00998.</span><span class="c2">&nbsp;Peter J. Liu, Yu-An Chung, and </span><span class="c1">Jie Ren. </span><span class="c2">[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/1910.00998&amp;sa=D&amp;source=editors&amp;ust=1702058950716606&amp;usg=AOvVaw0rdGCmkwUX-pnYbs11nXmQ">paper</a></span><span class="c5 c2">]</span></li><li class="c10 li-bullet-0"><span class="c2 c6">Can You Trust Your Model&rsquo;s Uncertainty? Evaluating Predictive Uncertainty Under Dataset Shift. </span><span class="c1">NeurIPS</span><span class="c2">&nbsp;(2019). &nbsp;Yaniv Ovadia, Emily Fertig, </span><span class="c1">Jie Ren</span><span class="c2">, Zachary Nado, David Sculley, Sebastian Nowozin, Joshua V. Dillon, Balaji Lakshminarayanan, and Jasper Snoek. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://arxiv.org/abs/1906.02530&amp;sa=D&amp;source=editors&amp;ust=1702058950716863&amp;usg=AOvVaw1H-bEyWwiwKVMesQVLjWTY">paper</a></span><span class="c2">] [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/google-research/google-research/tree/master/uq_benchmark_2019&amp;sa=D&amp;source=editors&amp;ust=1702058950716984&amp;usg=AOvVaw1bIy37y_MF5BHSwgASp-EU">code</a></span><span class="c2">] [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-benchmark-poster.pdf&amp;sa=D&amp;source=editors&amp;ust=1702058950717089&amp;usg=AOvVaw0h1O-Qg2tjT5vFWl5uDmY7">poster</a></span><span class="c2">] [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://ai.googleblog.com/2020/01/can-you-trust-your-models-uncertainty.html&amp;sa=D&amp;source=editors&amp;ust=1702058950717217&amp;usg=AOvVaw0nIZ_DyGCD6YX8F5b1KYF2">blog</a></span><span class="c5 c2">] &nbsp; &nbsp; </span></li></ul><p class="c12"><span class="c5 c2">&nbsp;</span></p><p class="c12"><span class="c8 c2 c6">Machine Learning in Biological Sciences</span></p><ul class="c14 lst-kix_tg2n18lwzkp6-0 start"><li class="c10 li-bullet-0"><span class="c2 c6">KIMI: Knockoff Inference for Motif Identification From Molecular Sequences With Controlled False Discovery Rate</span><span class="c2">. </span><span class="c1">Bioinformatics</span><span class="c2">&nbsp;37, no. 6 (2021): 759-766. Bai Xin, </span><span class="c1">Jie Ren</span><span class="c2">, Yingying Fan, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://academic.oup.com/bioinformatics/article-abstract/37/6/759/5942973&amp;sa=D&amp;source=editors&amp;ust=1702058950717746&amp;usg=AOvVaw1z1lFbzeCzVwFjO6v-Jb2I">paper]</a></span><span class="c5 c2">&nbsp;</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Identifying Viruses From Metagenomic Data Using Deep Learning. </span><span class="c1">Quantitative Biology </span><span class="c2">(2020): 1-14</span><span class="c1">. Jie Ren*, </span><span class="c2">Kai Song*, Chao Deng, Nathan A. Ahlgren, Jed A. Fuhrman, Yi Li, Xiaohui Xie, Ryan Poplin, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://link.springer.com/content/pdf/10.1007/s40484-019-0187-4.pdf&amp;sa=D&amp;source=editors&amp;ust=1702058950718030&amp;usg=AOvVaw1Yeh5DW2XAniP5EFdEl21u">paper</a></span><span class="c2">] [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/jessieren/DeepVirFinder&amp;sa=D&amp;source=editors&amp;ust=1702058950718121&amp;usg=AOvVaw1tEDaZ1uWilcxZmDv00Tg_">code</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">A Network-Based Integrated Framework for Predicting Virus&ndash;Prokaryote Interactions.</span><span class="c2">&nbsp;</span><span class="c1">NAR Genomics and Bioinformatics</span><span class="c2">&nbsp;2, no. 2 (2020): lqaa044. Weili Wang*, </span><span class="c1">Jie Ren*</span><span class="c2">, Kujin Tang, Emily Dart, Julio Cesar Ignacio-Espinoza, Jed A. Fuhrman, Jonathan Braun, Fengzhu Sun, and Nathan A. Ahlgren. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://academic.oup.com/nargab/article/2/2/lqaa044/5861484?login%3Dtrue&amp;sa=D&amp;source=editors&amp;ust=1702058950718402&amp;usg=AOvVaw2u-g_WB1t25l1CO9OZznXq">paper]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/jessieren/VirHostMatcher&amp;sa=D&amp;source=editors&amp;ust=1702058950718494&amp;usg=AOvVaw2k10NPM3duZoi09e_hJa3v">code</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Rapid Diagnosis and Comprehensive Bacteria Profiling of Sepsis Based on Cell-Free Dna. </span><span class="c1">Journal of Translational Medicine</span><span class="c2">&nbsp;18, no. 1 (2020): 1-10. Pei Chen, Shuo Li, Wenyuan Li, </span><span class="c1">Jie Ren</span><span class="c2">, Fengzhu Sun, Rui Liu, and Xianghong Jasmine Zhou. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://link.springer.com/article/10.1186/s12967-019-02186-x;&amp;sa=D&amp;source=editors&amp;ust=1702058950718774&amp;usg=AOvVaw0g7jxkoEnmXsrbBL7C_Fwg">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">Afann: Bias Adjustment for Alignment-Free Sequence Comparison Based on Sequencing Data Using Neural Network Regression.</span><span class="c2">&nbsp;</span><span class="c1">Genome Biology</span><span class="c2">&nbsp;20.1 (2019): 1-17</span><span class="c1">. </span><span class="c2">Kujin Tang,</span><span class="c1">&nbsp;Jie Ren</span><span class="c2">, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1872-3&amp;sa=D&amp;source=editors&amp;ust=1702058950719202&amp;usg=AOvVaw35Kkx4BN7EEGcJvqJ5Zta-">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Metagenomic Unmapped Reads Provide Important Insights Into Human Microbiota and Disease Associations. </span><span class="c1">Genome Biology</span><span class="c2">&nbsp;20, no. 1 (2019): 1-13. Zifan Zhu,</span><span class="c1">&nbsp;Jie Ren</span><span class="c2">, Sonia Michail, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1773-5&amp;sa=D&amp;source=editors&amp;ust=1702058950719511&amp;usg=AOvVaw0l23466ep_05VxspXezFz-">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Alignment-Free Oligonucleotide Frequency Dissimilarity Measure Improves Prediction of Hosts From Metagenomically-Derived Viral Sequences. </span><span class="c1">Nucleic Acids Research </span><span class="c2">45.1 (2018), 39-53</span><span class="c1">. </span><span class="c2">Nathan A. Ahlgren*, </span><span class="c1">Jie Ren*</span><span class="c2">, Yang Y. Lu, Jed A. Fuhrman, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://academic.oup.com/nar/article/45/1/39/2605663?login%3Dtrue&amp;sa=D&amp;source=editors&amp;ust=1702058950719803&amp;usg=AOvVaw3kK8ub5PwB6_hwIBSS0ks4">paper</a></span><span class="c2">] [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/jessieren/VirHostMatcher&amp;sa=D&amp;source=editors&amp;ust=1702058950719897&amp;usg=AOvVaw2oobmuJlw0yUAgPlHW6zDe">code</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">VirFinder: A Novel K-Mer Based Tool for Identifying Viral Sequences From Assembled Metagenomic Data. </span><span class="c1">Microbiome</span><span class="c2">&nbsp;5, no. 1 (2017): 1-20. </span><span class="c1">Jie Ren*, </span><span class="c2">Nathan A. Ahlgren*, Yang Young Lu, Jed A. Fuhrman, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://link.springer.com/article/10.1186/s40168-017-0283-5&amp;sa=D&amp;source=editors&amp;ust=1702058950720168&amp;usg=AOvVaw0O8lye5CKrn4MioVuBEjBh">paper]</a></span><span class="c2">&nbsp;[</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://github.com/jessieren/VirFinder&amp;sa=D&amp;source=editors&amp;ust=1702058950720256&amp;usg=AOvVaw1t82ZjQd5pZdmu-NTJqFyJ">code</a></span><span class="c2">] [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://anaconda.org/bioconda/r-virfinder&amp;sa=D&amp;source=editors&amp;ust=1702058950720341&amp;usg=AOvVaw1Q1ThU4Gb1Ino_jNiNvUns">R package</a></span><span class="c2">] [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.nature.com/articles/d41586-018-03358-3&amp;sa=D&amp;source=editors&amp;ust=1702058950720432&amp;usg=AOvVaw2lJJJJaWRy9wPeA_5pwlgT">news</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Identifying Group-Specific Sequences for Microbial Communities Using Long K-Mer Sequence Signatures. </span><span class="c1">Frontiers in Microbiology</span><span class="c2">&nbsp;9 (2018): 872. Ying Wang, Lei Fu, </span><span class="c1">Jie Ren</span><span class="c2">, Zhaoxia Yu, Ting Chen, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.frontiersin.org/articles/10.3389/fmicb.2018.00872/full&amp;sa=D&amp;source=editors&amp;ust=1702058950720740&amp;usg=AOvVaw2lu7MelzhIaSsPGqyDwvPs">paper</a></span><span class="c5 c2">]</span></li><li class="c10 li-bullet-0"><span class="c2 c6">Prediction of Virus-Host Infectious Association by Supervised Learning Methods. </span><span class="c1">BMC Bioinformatics</span><span class="c2">&nbsp;18.3 (2017): 143-154. Mengge Zhang, Lianping Yang, </span><span class="c1">Jie Ren</span><span class="c2">, Nathan A. Ahlgren, Jed A. Fuhrman, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1473-7&amp;sa=D&amp;source=editors&amp;ust=1702058950721002&amp;usg=AOvVaw1AQ_2bmHlMaZtEtdxM6Uti">paper</a></span><span class="c5 c2">]</span></li></ul><p class="c9"><span class="c8 c2 c6"></span></p><p class="c12"><span class="c8 c2 c6">Statistical Modeling in Genomics Data</span></p><ul class="c14 lst-kix_ibkhg9oor910-0 start"><li class="c10 li-bullet-0"><span class="c2 c6">A New Context Tree Inference Algorithm for Variable Length Markov Chain Model with Applications to Biological Sequence Analyses.</span><span class="c2">&nbsp;</span><span class="c1">Journal of Computational Biology</span><span class="c2">&nbsp;(2022). Shaokun An, </span><span class="c1">Jie Ren</span><span class="c2">, Fengzhu Sun, and Lin Wan. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0604&amp;sa=D&amp;source=editors&amp;ust=1702058950721349&amp;usg=AOvVaw3CVh0vZYsNflaOVQQLM1FP">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Confidence Intervals for Markov Chain Transition Probabilities Based on Next Generation Sequencing Reads Data.</span><span class="c2">&nbsp;</span><span class="c1">Quantitative Biology </span><span class="c2">8 (2020): 143-154</span><span class="c1">. </span><span class="c2">Lin Wan, Xin Kang, </span><span class="c1">Jie Ren</span><span class="c2">, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://link.springer.com/article/10.1007/s40484-020-0200-y&amp;sa=D&amp;source=editors&amp;ust=1702058950721644&amp;usg=AOvVaw1LPw1DApULJ723i3ZjuaFA">paper</a></span><span class="c5 c2">] </span></li><li class="c0 li-bullet-0"><span class="c2 c6">Predicting the Number of Bases to Attain Sufficient Coverage in High-Throughput Sequencing Experiments. </span><span class="c1">Journal of Computational Biology</span><span class="c2">&nbsp;27, no. 7 (2020): 1130-1143. Chao Deng, Timothy Daley, Peter Calabrese, </span><span class="c1">Jie Ren</span><span class="c2">, and Andrew D. Smith. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.liebertpub.com/doi/abs/10.1089/cmb.2019.0264&amp;sa=D&amp;source=editors&amp;ust=1702058950721887&amp;usg=AOvVaw1IpIC63EN7t6oEnZ_UI14p">paper]</a></span></li><li class="c0 li-bullet-0"><span class="c2 c6">Reads Binning Improves Alignment-Free Metagenome Comparison.</span><span class="c2">&nbsp;</span><span class="c1">Frontiers in Genetics</span><span class="c2">&nbsp;10 (2019): 1156. Kai Song, </span><span class="c1">Jie Ren</span><span class="c2">, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://internal-journal.frontiersin.org/articles/10.3389/fgene.2019.01156/full&amp;sa=D&amp;source=editors&amp;ust=1702058950722157&amp;usg=AOvVaw2cmWxZJzG3PnKar0L2x3Ss">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Alignment-Free Sequence Analysis and Applications. </span><span class="c1">Annual Review of Biomedical Data Science</span><span class="c2">&nbsp;1 (2018): 93-114. </span><span class="c1">Jie Ren</span><span class="c2">, Xin Bai, Yang Young Lu, Kujin Tang, Ying Wang, Gesine Reinert, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.annualreviews.org/doi/abs/10.1146/annurev-biodatasci-080917-013431&amp;sa=D&amp;source=editors&amp;ust=1702058950722414&amp;usg=AOvVaw3ElqeAOZiWROxj2APBQB_a">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Alignment-Free Genome Comparison Enables Accurate Geographic Sourcing of White Oak DNA. </span><span class="c1">BMC Genomics</span><span class="c2">&nbsp;19, no. 1 (2018): 1-16. Kujin Tang, </span><span class="c1">Jie Ren</span><span class="c2">, Richard Cronn, David L. Erickson, Brook G. Milligan, Meaghan Parker-Forney, John L. Spouge, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5253-1&amp;sa=D&amp;source=editors&amp;ust=1702058950722677&amp;usg=AOvVaw02U5lSo6RXIfEWhP7t8NWz">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Gut Microbial and Metabolomic Profiles After Fecal Microbiota Transplantation in Pediatric Ulcerative Colitis Patients.</span><span class="c2">&nbsp;</span><span class="c1">FEMS Microbiology Ecology</span><span class="c2">&nbsp;94, no. 9 (2018): fiy133. Nusbaum, David J., Fengzhu Sun,</span><span class="c1">&nbsp;Jie Ren</span><span class="c2">, Zifan Zhu, Natalie Ramsy, Nicholas Pervolarakis, Sachin Kunde et al. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://academic.oup.com/femsec/article/94/9/fiy133/5053801?login%3Dtrue&amp;sa=D&amp;source=editors&amp;ust=1702058950722948&amp;usg=AOvVaw3uZpPbiUSnUvUX86PWGJ5Z">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Optimal Choice of Word Length When Comparing Two Markov Sequences Using a &chi;2-Statistic. </span><span class="c1">BMC Genomics</span><span class="c2">&nbsp;18.6 (2017): 19-30. Xin Bai, Kujin Tang, </span><span class="c1">Jie Ren</span><span class="c2">, Michael Waterman, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://link.springer.com/article/10.1186/s12864-017-4020-z&amp;sa=D&amp;source=editors&amp;ust=1702058950723209&amp;usg=AOvVaw3HEABNbERkf28VwdsxB8Ig">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Cafe: Accelerated Alignment-Free Sequence Analysis. </span><span class="c1">Nucleic Acids Research</span><span class="c2">&nbsp;45.1 (2017): W554-W559. Yang Y. Lu, Kujin Tang, Jie Ren, Jed A. Fuhrman, Michael S. Waterman, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://academic.oup.com/nar/article/45/W1/W554/3791208?login%3Dtrue&amp;sa=D&amp;source=editors&amp;ust=1702058950723416&amp;usg=AOvVaw2MPRVpKUox-NrjMfUQhImj">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Alignment-Free Transcriptomic and Metatranscriptomic Comparison Using Sequencing Signatures With Variable Length Markov Chains. </span><span class="c1">Scientific Reports</span><span class="c2">&nbsp;6.1 (2016): 1-15. Weinan Liao*, </span><span class="c1">Jie Ren*</span><span class="c2">, Kun Wang, Shun Wang, Feng Zeng, Ying Wang, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.nature.com/articles/srep37243&amp;sa=D&amp;source=editors&amp;ust=1702058950723654&amp;usg=AOvVaw1X5I6qUnnFbhkRuoBaTObE">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Inference of Markovian Properties of Molecular Sequences From Ngs Data and Applications to Comparative Genomics. </span><span class="c1">Bioinformatics </span><span class="c2">32.7 (2016): 993-1000. </span><span class="c1">Jie Ren</span><span class="c2">, Kai Song, Minghua Deng, Gesine Reinert, Charles H. Cannon, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://academic.oup.com/bioinformatics/article/32/7/993/2240327?login%3Dtrue&amp;sa=D&amp;source=editors&amp;ust=1702058950723915&amp;usg=AOvVaw14KsJSDWUftamCadb5p3bE">paper</a></span><span class="c2 c5">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">New Developments of Alignment-Free Sequence Comparison: Measures, Statistics and Next-Generation Sequencing. </span><span class="c1">Briefings in Bioinformatics</span><span class="c2">&nbsp;15.3 (2014): 343-353. Kai Song, </span><span class="c1">Jie Ren</span><span class="c2">, Gesine Reinert, Minghua Deng, Michael S. Waterman, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://academic.oup.com/bib/article/15/3/343/182355?login%3Dtrue&amp;sa=D&amp;source=editors&amp;ust=1702058950724257&amp;usg=AOvVaw0LB6EYCRYWaWUARMeJ7J9x">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Multiple Alignment-Free Sequence Comparison. </span><span class="c1">Bioinformatics 29.21 (2013): 2690-2698. Jie Ren</span><span class="c2">, Kai Song, Fengzhu Sun, Minghua Deng, and Gesine Reinert. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://academic.oup.com/bioinformatics/article/29/21/2690/195361?login%3Dtrue&amp;sa=D&amp;source=editors&amp;ust=1702058950724534&amp;usg=AOvVaw12fWZ2oE8xKtjJgbSd7z_Q">paper</a></span><span class="c5 c2">]</span></li><li class="c0 li-bullet-0"><span class="c2 c6">Alignment-Free Sequence Comparison Based on Next-Generation Sequencing Reads. </span><span class="c1">Journal of Computational Biology</span><span class="c2">&nbsp;20.2 (2013): 64-79. Kai Song, </span><span class="c1">Jie Ren,</span><span class="c2">&nbsp;Zhiyuan Zhai, Xuemei Liu, Minghua Deng, and Fengzhu Sun. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.liebertpub.com/doi/abs/10.1089/cmb.2012.0228&amp;sa=D&amp;source=editors&amp;ust=1702058950724831&amp;usg=AOvVaw3mVT-kyl_rWTpqF7PC-0CO">paper</a></span><span class="c5 c2">]</span></li><li class="c10 li-bullet-0"><span class="c2 c6">Comparison of Metagenomic Samples Using Sequence Signatures. </span><span class="c1">BMC Genomics</span><span class="c2">&nbsp;13.1 (2012): 1-17.</span><span class="c2 c6">&nbsp;</span><span class="c2">Bai Jiang*, Kai Song*, </span><span class="c1">Jie Ren*</span><span class="c2">, Minghua Deng, Fengzhu Sun, and Xuegong Zhang. [</span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-13-730&amp;sa=D&amp;source=editors&amp;ust=1702058950725186&amp;usg=AOvVaw0Hgx0fYJpHs11g2fbNSwar">paper</a></span><span class="c5 c2">]</span></li></ul><p class="c12"><span class="c5 c2">&nbsp;</span></p><p class="c12"><span class="c8 c2 c6">Talks</span></p><ul class="c14 lst-kix_d1ebcswm62mz-0 start"><li class="c10 li-bullet-0"><span class="c2">2023 </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=http://wids-santaclara.org/?i%3D1&amp;sa=D&amp;source=editors&amp;ust=1702058950725453&amp;usg=AOvVaw0QqXd9ymhjwFLtfeU8JfVd">Women in Data Science SANTA CLARA 2023</a></span><span class="c5 c2">: Building neural networks that know what they don&#39;t know.</span></li><li class="c0 li-bullet-0"><span class="c2">2023 </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://sites.google.com/view/vand-cvpr23/home&amp;sa=D&amp;source=editors&amp;ust=1702058950725623&amp;usg=AOvVaw2O4KFkm3o4xsBaOe2vuKKQ">Visual Anomaly and Novelty Detection Workshop</a></span><span class="c5 c2">&nbsp;@ CVPR 2023: OOD detection and uncertainty estimation using text-image models.</span></li><li class="c0 li-bullet-0"><span class="c5 c2">2022 Amazon BioSciences Colloquium: Does Your Model Know What It Doesn&rsquo;t Know? Exploring the Limits of Out-of-distribution Detection.</span></li><li class="c0 li-bullet-0"><span class="c2">2022 UCL Centre for Artificial Intelligence: </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.youtube.com/watch?v%3D8cdBWK4Y45s&amp;sa=D&amp;source=editors&amp;ust=1702058950725831&amp;usg=AOvVaw3FIv3acWNgP2MKenIT9jxd">Reliable Deep Anomaly Detection</a></span></li><li class="c0 li-bullet-0"><span class="c2">2021 Anomaly Detection for Scientific Discovery Seminars: </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://ad4sd.github.io/seminars/&amp;sa=D&amp;source=editors&amp;ust=1702058950725991&amp;usg=AOvVaw3bMW57FGxqQGSu2dbzPr1i">Exploring the limits of out-of-distribution detection in vision and biomedical applications</a></span></li><li class="c0 li-bullet-0"><span class="c2">2021 ICML WiML Un-Workshop: </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://docs.google.com/presentation/d/1E90g5yJpYZBa-_jRYGc0IMSnE0axf0wHskUKS1XD-9Y/edit?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1702058950726248&amp;usg=AOvVaw1OzShMNb5iWojx_gFnqRnR">Uncertainty Estimation and Out-of-Distribution (Ood) Detection in Deep Learning</a></span></li><li class="c0 li-bullet-0"><span class="c2">2019 NeurIPS: </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://www.youtube.com/watch?v%3D-FduW9ZWAR4&amp;sa=D&amp;source=editors&amp;ust=1702058950726449&amp;usg=AOvVaw1LlzwGVPnvnwEKz_4ESmcL">Likelihood Ratios for Out-of-Distribution Detection.</a></span></li><li class="c0 li-bullet-0"><span class="c5 c2">2016 Molecular and Computational Biology Retreat, University of Southern California: Predicting Virus-Host Interactions Using Sequence Signatures.</span></li><li class="c0 li-bullet-0"><span class="c5 c2">2015 Joint USC/UCLA Bioinformatics Colloquium: Statistical Inference of Markov Models of Genomic Sequences Using NGS Data.</span></li><li class="c0 li-bullet-0"><span class="c5 c2">2015 Research in Computational Molecular Biology (RECOMB)-Seq: Inference of Markovian Properties of Molecular Sequences From NGS Data and Applications to Comparative Genomics.</span></li><li class="c10 li-bullet-0"><span class="c5 c2">2013 Molecular and Computational Biology Retreat, University of Southern California: Alignment-Free Genome and Metagenome Comparison Based on Next Generation Sequencing Reads.</span></li></ul><p class="c12"><span class="c5 c2">&nbsp;</span></p><p class="c12"><span class="c8 c2 c6">Mentorship and Community</span></p><ul class="c14 lst-kix_9rbk2sol80ch-0 start"><li class="c10 li-bullet-0"><span class="c5 c2">Interns: Yunhao Ge (2022), James Allingham (2022), Kundan Krishna (2022), Yu-An Chung (2019).</span></li><li class="c0 li-bullet-0"><span class="c5 c2">Invited reviewer for machine learning venues: NeurIPS (2019-), ICML (2019-), ICLR (2019-), IEEE Transactions on Artificial Intelligence </span></li><li class="c0 li-bullet-0"><span class="c5 c2">Invited reviewer for bioinformatics journals: Nature Microbiology, Cell Systems, Medical Image Analysis, Nucleic Acids Research, Gut, Bioinformatics, Giga Science, Scientific Reports, PLOS Computational Biolofy, PLOS ONE. </span></li><li class="c10 li-bullet-0"><span class="c5 c2">Program Committee (PC) for ICML workshop on Uncertainty and Robustness in Deep Learning &nbsp;(2020-2021) </span></li></ul><p class="c12"><span class="c5 c2">&nbsp;</span></p><p class="c12"><span class="c2 c6 c8">Education</span></p><ul class="c14 lst-kix_6ftjd98kskfz-0 start"><li class="c10 li-bullet-0"><span class="c5 c2">Postdoc Fellow, University of Southern California, Los Angeles, USA. (2017-2018).</span></li><li class="c0 li-bullet-0"><span class="c5 c2">Ph.D. in Computational Biology and Bioinformatics, University of Southern California, Los Angeles, USA. (2013-2017).</span></li></ul><ul class="c14 lst-kix_6ftjd98kskfz-1 start"><li class="c20 li-bullet-0"><span class="c2">Advisors: </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://dornsife.usc.edu/cf/faculty-and-staff/faculty.cfm?pid%3D1003742&amp;sa=D&amp;source=editors&amp;ust=1702058950727459&amp;usg=AOvVaw3ihBBhgaJK1mmqMSQkkMEb">Fengzhu Sun</a></span><span class="c2">, </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://dornsife.usc.edu/cf/faculty-and-staff/faculty.cfm?pid%3D1003269&amp;sa=D&amp;source=editors&amp;ust=1702058950727587&amp;usg=AOvVaw3rWBfImT7oo6OEHnGQZXAa">Jed Fuhrman</a></span><span class="c2">, </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=https://dornsife.usc.edu/labs/msw/&amp;sa=D&amp;source=editors&amp;ust=1702058950727684&amp;usg=AOvVaw1HvLmviXVf0O5JakDTMDZ_">Michael Waterman</a></span><span class="c2">, </span><span class="c3"><a class="c4" href="https://www.google.com/url?q=http://www.stats.ox.ac.uk/~reinert/&amp;sa=D&amp;source=editors&amp;ust=1702058950727776&amp;usg=AOvVaw22nezmiuHRMaiYoqzbt-p1">Gesine Reinert</a></span></li></ul><ul class="c14 lst-kix_6ftjd98kskfz-0"><li class="c0 li-bullet-0"><span class="c5 c2">M.S. in Statistics, University of Southern California, Los Angeles, USA. (2016-2017).</span></li><li class="c0 li-bullet-0"><span class="c5 c2">Exchange Student in Bioinformatics, Stockholm University, Stockholm, Sweden. (2011).</span></li><li class="c0 li-bullet-0"><span class="c5 c2">M.S. in Probability and Statistics, Peking University, Beijing, China. (2010-2013).</span></li><li class="c0 li-bullet-0"><span class="c5 c2">Double Major in Economics, Peking University, Beijing, China. (2009-2011).</span></li><li class="c0 li-bullet-0"><span class="c5 c2">Exchange Student in Mathematics, Hong Kong Baptist University, Hong Kong SAR. (2008).</span></li><li class="c10 li-bullet-0"><span class="c5 c2">B.S. in Information and Computing Science, Minzu University, Beijing, China. (2006-2010).</span></li></ul><p class="c13"><span class="c5 c2">&nbsp;</span></p><p class="c13"><span class="c5 c2">&nbsp;</span></p><p class="c16 c22"><span class="c5 c24"></span></p></body></html>