<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Jessie Jie Ren</title>

<!-- Global site tag (gtag.js) - Google Analytics -->
<!-- Yes yes I know, this totally ruins my "super 1 page website!!1" sell sad :'( -->
<script async="" src="https://www.googletagmanager.com/gtag/js?id=G-RZD43RNQPY"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'G-RZD43RNQPY');
</script>

<style>
/* inline css to save number of files. one file suffices! */
body {
    margin: 0;
    padding: 15px;
    font-size: 16px;
    line-height: 1.15;
    color: #111;
}
#wrap {
    max-width: 800px;
    margin-left: auto;
    margin-right: auto;
}
h1 {
    margin: 0px;
}
#links a {
    margin-right: 5px;
}
ul {
    padding-left: 20px;
}
.project {
    margin-top: 10px;
}
.pub {
    margin-top: 10px;
}
.pub-venue {
    display: inline;
    font-weight: bold;
}
.pub-authors {
    display: inline;
    font-style: italic;
}
.nodot {
  list-style-type: none;
  padding-left: 0;
}
.bdiv {
    margin-top: 5px;
}
#demail {
    margin-top: 10px;
    color: #555;
    opacity: 0;
    transition: opacity 0.7s;
}
</style>
</head>
<body>

<div id="wrap">
<h1>Jessie Jie Ren</h1>
<div>Computational Biology and Bioinformatics; Machine learning; Statistical Modeling</div>
<div id="links"><a href="https://twitter.com/jessierenjie">Twitter</a> <a href="https://github.com/jessieren">Github</a> <a href="https://scholar.google.com/citations?user=Os9wmpkAAAAJ&amp;hl=en">Google Scholar</a> <span id="iemail" style="cursor: pointer; text-decoration: underline;">Email</span></div>
<div id="demail">&nbsp;</div>
<!-- will reveal email --> <br />
<div>I am a Research Scientist in Google Research Brain Team. I received my PhD in Computational Biology and Bioinformatics and MSc in Statistics at the University of Southern California. The focus of my research has been machine learning and statistical modeling in biological and medical problems, uncertainty estimation and out-of-distribution detection in deep learning.</div>
<div>&nbsp;</div>
<h2>News</h2>
<ul>
<li>07/2021: Together with <a href="https://polkirichenko.github.io/">Polina Kirichenko</a>, <a href="https://scholar.google.com/citations?user=-SjNMu8AAAAJ&amp;hl=en">Haleh Akrami</a>, <a href="http://pages.cs.wisc.edu/~sharonli/">Sharon Yixuan Li</a> and <a href="https://scholar.google.com/citations?user=D_CuHyUAAAAJ&amp;hl=en">Serg&uuml;l Ayd&ouml;re</a>, we are going to lead a <a href="https://icml.cc/virtual/2021/affinityworkshop/10849">breakout session</a> at July 21 7:25 pm ET at WiML Un-Workshop at ICML 2021 on "Does your model know what it doesn&rsquo;t know? Uncertainty estimation and OOD detection in deep learning."</li>
<li>07/2021: Our paper "A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection" (<a href="https://arxiv.org/abs/2106.09022">paper</a>, <a href="https://twitter.com/jessierenjie/status/1418321220546613248">poster</a>) will be presented at ICML 2021 UDL workshop.</li>
<li>07/2021: Our paper "Exploring the Limits of Out-of-Distribution Detection" (<a href="https://arxiv.org/abs/2106.03004">paper</a>) will be presented at ICML 2021 UDL workshop.</li>
</ul>
<p>&nbsp;</p>
<h2>News</h2>
<ul>
<li>07/2021: Together with <a href="https://polkirichenko.github.io/">Polina Kirichenko</a>, <a href="https://scholar.google.com/citations?user=-SjNMu8AAAAJ&amp;hl=en">Haleh Akrami</a>, <a href="http://pages.cs.wisc.edu/~sharonli/">Sharon Yixuan Li</a> and <a href="https://scholar.google.com/citations?user=D_CuHyUAAAAJ&amp;hl=en">Serg&uuml;l Ayd&ouml;re</a>, we are going to lead a <a href="https://icml.cc/virtual/2021/affinityworkshop/10849">breakout session</a> at July 21 7:25 pm ET at WiML Un-Workshop at ICML 2021 on "Does your model know what it doesn&rsquo;t know? Uncertainty estimation and OOD detection in deep learning."</li>
<li>07/2021: Our paper "A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection" (<a href="https://arxiv.org/abs/2106.09022">paper</a>, <a href="https://twitter.com/jessierenjie/status/1418321220546613248">poster</a>) will be presented at ICML 2021 UDL workshop.</li>
<li>07/2021: Our paper "Exploring the Limits of Out-of-Distribution Detection" (<a href="https://arxiv.org/abs/2106.03004">paper</a>) will be presented at ICML 2021 UDL workshop.</li>
<li>06/2019: I am joining Google Brain as a Research Scientist.&nbsp;</li>
<li>06/2018: I am joining Google Brain AI Residency program.&nbsp;</li>
<li>03/2018: Our work is highlighted in <em>Nature News </em>titled "<a href="https://www.nature.com/articles/d41586-018-03358-3">Machine learning spots treasure trove of elusive viruses Artificial intelligence could speed up metagenomic studies that look for species unknown to science.</a>".</li>
</ul>
<p>&nbsp;</p>
<h2>Publications</h2>
<h4>Uncertainty Estimation and Out-of-distribution Detection</h4>
<ul>
<li><strong style="font-size: 14px;">A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection.&nbsp;</strong><strong style="font-size: 14px;"><em style="font-weight: 400;">arXiv preprint arXiv:2106.09022</em><span style="font-weight: 400;">&nbsp;(2021).&nbsp;</span></strong><em style="font-size: 14px;">Jie Ren</em><span style="font-size: 14px;">, Stanislav Fort, Jeremiah Liu, Abhijit Guha Roy, Shreyas Padhy, and Balaji Lakshminarayanan. [<a href="https://arxiv.org/abs/2106.09022">paper]&nbsp;</a><a href="https://twitter.com/jessierenjie/status/1418321220546613248">[poster]</a></span></li>
</ul>
<div class="pub">
<ul>
<li class="pub-title">
<p><strong>Exploring the Limits of Out-of-Distribution Detection</strong>.&nbsp;<em>arXiv preprint arXiv:2106.03004</em>&nbsp;(2021). Stanislav Fort*, <em>Jie Ren*</em>, and Balaji Lakshminarayanan.[<a href="https://arxiv.org/abs/2106.03004">paper]</a>&nbsp; &nbsp; &nbsp;&nbsp;</p>
</li>
<li class="pub-title">
<p><strong>Does Your Dermatology Classifier Know What It Doesn't Know? Detecting the Long-Tail of Unseen Conditions.</strong>&nbsp;<em>arXiv preprint arXiv:2104.03829</em>&nbsp;(2021). Abhijit Guha Roy*, <em>Jie Ren*</em>, Shekoofeh Azizi, Aaron Loh, Vivek Natarajan, Basil Mustafa, Nick Pawlowski et al.&nbsp;[<a href="https://arxiv.org/abs/2104.03829">paper]</a>&nbsp;</p>
</li>
<li>
<p><strong>Uncertainty Baselines: Benchmarks for Uncertainty &amp; Robustness in Deep Learning.&nbsp;</strong><em>arXiv preprint arXiv:2106.04015</em>&nbsp;(2021). Zachary Nado, Neil Band, Mark Collier, Josip Djolonga, Michael W. Dusenberry, Sebastian Farquhar, Angelos Filos et al.&nbsp;[<a href="https://arxiv.org/abs/2106.04015">paper]</a></p>
</li>
<li><strong>Revisiting one-vs-all classifiers for predictive uncertainty and out-of-distribution detection in neural networks.</strong>&nbsp;<em>arXiv preprint arXiv:2007.05134</em>&nbsp;(2020). Shreyas Padhy, Zachary Nado, <em>Jie Ren,</em> Jeremiah Liu, Jasper Snoek, and Balaji Lakshminarayanan. [<a href="https://arxiv.org/abs/2007.05134">paper]</a>&nbsp; &nbsp; &nbsp; &nbsp;</li>
</ul>
<ul>
<li class="pub-title"><strong>Likelihood ratios for out-of-distribution detection. </strong><em>NeurIPS (2019)</em>.&nbsp;<em>Jie Ren</em>, Peter J. Liu, Emily Fertig, Jasper Snoek, Ryan Poplin, Mark A. DePristo, Joshua V. Dillon, and Balaji Lakshminarayanan.&nbsp;[<a href="https://arxiv.org/abs/1906.02845">paper] [</a><a href="https://github.com/google-research/google-research/tree/master/genomics_ood">code] [</a><a href="http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-llr-ood-poster.pdf">poster] [</a><a href="https://www.youtube.com/watch?v=-FduW9ZWAR4">3-minute video]&nbsp;</a>[<a href="https://ai.googleblog.com/2019/12/improving-out-of-distribution-detection.html">blog]</a></li>
<li>
<p><strong>Can you trust your model's uncertainty? Evaluating predictive uncertainty under dataset shift.&nbsp;&nbsp;</strong><em>arXiv preprint arXiv:1906.02530</em>&nbsp;(2019). Yaniv Ovadia, Emily Fertig, <em>Jie Ren</em>, Zachary Nado, David Sculley, Sebastian Nowozin, Joshua V. Dillon, Balaji Lakshminarayanan, and Jasper Snoek.&nbsp;[<a href="https://arxiv.org/abs/1906.02530">paper</a>] [<a href="https://github.com/google-research/google-research/tree/master/uq_benchmark_2019">code</a>] [<a href="http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-benchmark-poster.pdf">poster</a>] [<a href="https://ai.googleblog.com/2020/01/can-you-trust-your-models-uncertainty.html">blog</a>]&nbsp; &nbsp; &nbsp;</p>
</li>
</ul>
<p>&nbsp;</p>
<h4>Bioinformatics</h4>
<ul>
<li><strong>KIMI: Knockoff Inference for Motif Identification from molecular sequences with controlled false discovery rate</strong>. <em>Bioinformatics</em>&nbsp;37, no. 6 (2021): 759-766. Bai Xin, <em>Jie Ren</em>, Yingying Fan, and Fengzhu Sun. [<a href="https://academic.oup.com/bioinformatics/article-abstract/37/6/759/5942973">paper]</a>&nbsp;</li>
</ul>
<ul>
<li>
<p><strong>Rapid diagnosis and comprehensive bacteria profiling of sepsis based on cell-free DNA.</strong>&nbsp;<em>Journal of translational medicine</em>&nbsp;18, no. 1 (2020): 1-10. Pei Chen, Shuo Li, Wenyuan Li, <em>Jie Ren</em>, Fengzhu Sun, Rui Liu, and Xianghong Jasmine Zhou. [<a href="https://link.springer.com/article/10.1186/s12967-019-02186-x;">paper]</a>&nbsp;</p>
</li>
</ul>
<ul>
<li class="pub-title"><strong>Predicting the Number of Bases to Attain Sufficient Coverage in High-Throughput Sequencing Experiments. </strong>[<a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2019.0264">paper</a>]<strong>&nbsp;</strong><em>Journal of Computational Biology</em>&nbsp;27, no. 7 (2020): 1130-1143. Chao Deng, Timothy Daley, Peter Calabrese, <em>Jie Ren</em>, and Andrew D. Smith.&nbsp;[<a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2019.0264">paper]</a>&nbsp;</li>
<li class="pub-title">
<p><strong>A network-based integrated framework for predicting virus&ndash;prokaryote interactions.</strong>&nbsp;<em>NAR genomics and bioinformatics</em>&nbsp;2, no. 2 (2020): lqaa044. Weili Wang*, <em>Jie Ren*</em>, Kujin Tang, Emily Dart, Julio Cesar Ignacio-Espinoza, Jed A. Fuhrman, Jonathan Braun, Fengzhu Sun, and Nathan A. Ahlgren.&nbsp;[<a href="https://academic.oup.com/nargab/article/2/2/lqaa044/5861484?login=true">paper]</a>&nbsp;[<a href="https://github.com/jessieren/VirHostMatcher">code</a>]</p>
</li>
<li><strong>Confidence intervals for Markov chain transition probabilities based on next generation sequencing reads data.</strong>&nbsp;<em>Quantitative Biology 8 (2020), 143-154.&nbsp;</em>Lin Wan, Xin Kang, <em>Jie Ren</em>, Fengzhu Sun. [<a href="https://link.springer.com/content/pdf/10.1007/s40484-020-0200-y.pdf">paper</a>]&nbsp;</li>
<li>
<p><strong>Identifying viruses from metagenomic data using deep learning.&nbsp;</strong><em>Quantitative Biology (2020), 1-14.&nbsp;Jie Ren*, </em>Kai Song*, Chao Deng, Nathan A. Ahlgren, Jed A. Fuhrman, Yi Li, Xiaohui Xie, Ryan Poplin, Fengzhu Sun. [<a href="https://link.springer.com/content/pdf/10.1007/s40484-019-0187-4.pdf">paper</a>] [<a href="https://github.com/jessieren/DeepVirFinder">code</a>]</p>
</li>
<li>
<p><strong>Afann: bias adjustment for alignment-free sequence comparison based on sequencing data using neural network regression.</strong>&nbsp;<em>Genome biology 20 (1) (2019), 1-17.&nbsp;</em>Kujin Tang,<em> Jie Ren</em>, Fengzhu Sun. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1872-3">paper</a>]</p>
</li>
<li class="pub-title">
<p><strong>Metagenomic unmapped reads provide important insights into human microbiota and disease associations.&nbsp;</strong><em>Genome biology</em>&nbsp;20, no. 1 (2019): 1-13. Zifan Zhu,<em> Jie Ren</em>, Sonia Michail, Fengzhu Sun. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1773-5">paper</a>]</p>
</li>
<li>
<p><strong>VirFinder: a novel k-mer based tool for identifying viral sequences from assembled metagenomic data.&nbsp;</strong><em>Microbiome</em>&nbsp;5, no. 1 (2017): 1-20. <em>Jie Ren* </em>Nathan A. Ahlgren*, Yang Young Lu, Jed A. Fuhrman, Fengzhu Sun.&nbsp;[<a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=Os9wmpkAAAAJ&amp;citation_for_view=Os9wmpkAAAAJ:Vno172sVVMwC">paper]</a>&nbsp;[<a href="https://github.com/jessieren/VirFinder">code</a>] [<a href="https://anaconda.org/bioconda/r-virfinder">R package</a>] [<a href="https://www.nature.com/articles/d41586-018-03358-3">news</a>]</p>
</li>
<li class="pub-title"><strong>Reads binning improves alignment-free metagenome comparison.</strong>&nbsp;<em>Frontiers in genetics</em>&nbsp;10 (2019): 1156. Kai Song, <em>Jie Ren</em>, and Fengzhu Sun. [<a href="https://internal-journal.frontiersin.org/articles/10.3389/fgene.2019.01156/full">paper</a>]</li>
<li>
<p><strong>Alignment-free genome comparison enables accurate geographic sourcing of white oak DNA.&nbsp;</strong><em>BMC genomics</em>&nbsp;19, no. 1 (2018): 1-16. Kujin Tang, <em>Jie Ren</em>, Richard Cronn, David L. Erickson, Brook G. Milligan, Meaghan Parker-Forney, John L. Spouge, and Fengzhu Sun. [<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5253-1">paper</a>]</p>
</li>
<li><strong>Gut microbial and metabolomic profiles after fecal microbiota transplantation in pediatric ulcerative colitis patients</strong>.&nbsp;<em>FEMS microbiology ecology</em>&nbsp;94, no. 9 (2018): fiy133.&nbsp;Nusbaum, David J., Fengzhu Sun,<em> Jie Ren</em>, Zifan Zhu, Natalie Ramsy, Nicholas Pervolarakis, Sachin Kunde et al. [<a href="https://academic.oup.com/femsec/article/94/9/fiy133/5053801?login=true">paper</a>]</li>
<li>
<p><strong>Alignment-free sequence analysis and applications.&nbsp;</strong><em>Annual Review of Biomedical Data Science</em>&nbsp;1 (2018): 93-114. <em>Jie Ren</em>, Xin Bai, Yang Young Lu, Kujin Tang, Ying Wang, Gesine Reinert, and Fengzhu Sun. [<a href="https://www.annualreviews.org/doi/abs/10.1146/annurev-biodatasci-080917-013431">paper</a>]</p>
</li>
<li><strong>Identifying group-specific sequences for microbial communities using long k-mer sequence signatures.&nbsp;</strong><em>Frontiers in microbiology</em>&nbsp;9 (2018): 872. Ying Wang, Lei Fu, <em>Jie Ren</em>, Zhaoxia Yu, Ting Chen, and Fengzhu Sun. [<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.00872/full">paper</a>]</li>
</ul>
<h4>&nbsp;</h4>
<h4>Natural Language Processing</h4>
<ul>
<li><strong>SummAE: Zero-shot abstractive text summarization using length-agnostic auto-encoders.</strong>&nbsp;<em>arXiv preprint arXiv:1910.00998.</em> Peter J. Liu, Yu-An Chung, <em>Jie Ren. [<a href="https://arxiv.org/abs/1910.00998">paper</a>]</em></li>
</ul>
</div>
<h2>Talks</h2>
<ul class="pub">
<li>2021 ICML WiML Un-Workshop: <a href="https://docs.google.com/presentation/d/1E90g5yJpYZBa-_jRYGc0IMSnE0axf0wHskUKS1XD-9Y/edit?usp=sharing">Uncertainty estimation and out-of-distribution (OOD) detection in deep learning</a></li>
<li>2019 NeurIPS: <a href="https://www.youtube.com/watch?v=-FduW9ZWAR4">Likelihood ratios for out-of-distribution detection</a></li>
<li>2016 Molecular and Computational Biology Retreat, University of Southern California: Predicting virus-host interactions using sequence signatures.</li>
<li>2015 Joint USC/UCLA Bioinformatics Colloquium: Statistical inference of Markov models of genomic sequences using NGS data</li>
<li>2015 Research in Computational Molecular Biology (RECOMB)-Seq: Inference of Markovian properties of molecular sequences from NGS data and applications to comparative genomics</li>
<li>2013 Molecular and Computational Biology Retreat, University of Southern California: Alignment-free Genome and Metagenome Comparison Based On Next Generation Sequencing Reads</li>
</ul>
<p>&nbsp;</p>
</div>

<!-- attempt hiding from spambots :p -->
<script type="text/javascript">
var e_is_shown = false;
document.getElementById('iemail').addEventListener("click", function(){
    let demail = document.getElementById('demail');
    demail.innerHTML = 'renj' + ' _at_ ' + 'usc' + '.edu';
    demail.style.opacity = e_is_shown ? 0 : 1;
    e_is_shown = !e_is_shown;
})
</script>

</body>
</html>
