<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Jessie Jie Ren</title>

<!-- Global site tag (gtag.js) - Google Analytics -->
<!-- Yes yes I know, this totally ruins my "super 1 page website!!1" sell sad :'( -->
<script async="" src="https://www.googletagmanager.com/gtag/js?id=G-RZD43RNQPY"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'G-RZD43RNQPY');
</script>

<style>
/* inline css to save number of files. one file suffices! */
body {
    margin: 0;
    padding: 15px;
    font-size: 16px;
    line-height: 1.15;
    color: #111;
}
#wrap {
    max-width: 800px;
    margin-left: auto;
    margin-right: auto;
}
h1 {
    margin: 0px;
}
#links a {
    margin-right: 5px;
}
ul {
    padding-left: 20px;
}
.project {
    margin-top: 10px;
}
.pub {
    margin-top: 10px;
}
.pub-venue {
    display: inline;
    font-weight: bold;
}
.pub-authors {
    display: inline;
    font-style: italic;
}
.nodot {
  list-style-type: none;
  padding-left: 0;
}
.bdiv {
    margin-top: 5px;
}
#demail {
    margin-top: 10px;
    color: #555;
    opacity: 0;
    transition: opacity 0.7s;
}
</style>
</head>
<body>

<div id="wrap">
<div id="links">&nbsp;</div>
<h1>Jessie Jie Ren</h1>
<h1>&nbsp;<img src="https://scholar.googleusercontent.com/citations?view_op=medium_photo&amp;user=Os9wmpkAAAAJ&amp;citpid=10" alt="" width="120" height="120" /></h1>
<div>Computational Biology and Bioinformatics; Machine learning; Statistical Modeling</div>
<div id="links"><a href="https://twitter.com/jessierenjie">Twitter</a> <a href="https://github.com/jessieren">Github</a> <a href="https://scholar.google.com/citations?user=Os9wmpkAAAAJ&amp;hl=en">Google Scholar</a> <span id="iemail" style="cursor: pointer; text-decoration: underline;">Email</span></div>
<div id="demail">&nbsp;</div>
<!-- will reveal email -->
<div>I am a Research Scientist in <a href="https://research.google/teams/brain/">Google Research Brain Team</a>. Previously, I received my PhD in <a href="https://www.qcb-dornsife.usc.edu/">Computational Biology and Bioinformatics</a> and MSc in Statistics at the University of Southern California. The focus of my research has been machine learning and statistical modeling in biological and medical problems, uncertainty estimation and out-of-distribution detection in deep learning.</div>
<div>&nbsp;</div>
<h2>News</h2>
<ul>
<li>07/2021: Together with <a href="https://polkirichenko.github.io/">Polina Kirichenko</a>, <a href="https://scholar.google.com/citations?user=-SjNMu8AAAAJ&amp;hl=en">Haleh Akrami</a>, <a href="http://pages.cs.wisc.edu/~sharonli/">Sharon Yixuan Li</a> and <a href="https://scholar.google.com/citations?user=D_CuHyUAAAAJ&amp;hl=en">Serg&uuml;l Ayd&ouml;re</a>, we are going to lead a <a href="https://icml.cc/virtual/2021/affinityworkshop/10849">breakout session</a> at July 21 7:25 pm ET at WiML Un-Workshop at ICML 2021 on "Does your model know what it doesn&rsquo;t know? Uncertainty estimation and OOD detection in deep learning."</li>
<li>07/2021: Our paper "A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection" (<a href="https://arxiv.org/abs/2106.09022">paper</a>, <a href="https://twitter.com/jessierenjie/status/1418321220546613248">poster</a>) will be presented at ICML 2021 UDL workshop.</li>
<li>07/2021: Our paper "Exploring the Limits of Out-of-Distribution Detection" (<a href="https://arxiv.org/abs/2106.03004">paper</a>) will be presented at ICML 2021 UDL workshop.</li>
<li>06/2019: I am joining Google Brain as a Research Scientist.&nbsp;</li>
<li>06/2018: I am joining Google Brain AI Residency program.&nbsp;</li>
<li>03/2018: Our work is highlighted in <a href="https://www.nature.com/articles/d41586-018-03358-3"><em>Nature News </em></a>titled "Machine learning spots treasure trove of elusive viruses Artificial intelligence could speed up metagenomic studies that look for species unknown to science.".</li>
<li>04/2017: I am honored to receive the prestigious <a href="https://twitter.com/uscbiosci/status/858070439435710464">USC PhD Achievement Award</a> (6 winners university-wide per year).</li>
<li>03/2017: I am honored to receive Women in Science and Engineering Travel Grant.&nbsp;</li>
<li>04/2016: I am honored to receive <a href="https://dornsife.usc.edu/cams/cams-prize/">USC CAMS Graduate Student Prize</a> for Excellence in Research with a Substantial Mathematical Component (2 winners university-wide per year).&nbsp;</li>
</ul>
<p>&nbsp;</p>
<h2>Publications</h2>
<h4>Uncertainty Estimation and Out-of-distribution Detection in Deep Learning</h4>
<ul>
<li class="pub-title">
<p><strong>Exploring the Limits of Out-of-Distribution Detection</strong>.&nbsp;<em>arXiv preprint arXiv:2106.03004</em>&nbsp;(2021). Stanislav Fort*, <em>Jie Ren*</em>, and Balaji Lakshminarayanan.[<a href="https://arxiv.org/abs/2106.03004">paper]</a>&nbsp; &nbsp;</p>
</li>
</ul>
<ul>
<li><strong>A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection.&nbsp;</strong><em>arXiv preprint </em><em>arXiv:2106.09022</em>&nbsp;(2021)<strong>.&nbsp;</strong><em>Jie Ren</em>, Stanislav Fort, Jeremiah Liu, Abhijit Guha Roy, Shreyas Padhy, and Balaji Lakshminarayanan. [<a href="https://arxiv.org/abs/2106.09022">paper]&nbsp;</a><a href="https://twitter.com/jessierenjie/status/1418321220546613248">[poster]</a></li>
</ul>
<ul>
<li class="pub-title">
<p><strong>Does Your Dermatology Classifier Know What It Doesn't Know? Detecting the Long-Tail of Unseen Conditions.</strong>&nbsp;<em>arXiv preprint arXiv:2104.03829</em>&nbsp;(2021). Abhijit Guha Roy*, <em>Jie Ren*</em>, Shekoofeh Azizi, Aaron Loh, Vivek Natarajan, Basil Mustafa, Nick Pawlowski et al.&nbsp;[<a href="https://arxiv.org/abs/2104.03829">paper]</a>&nbsp;</p>
</li>
<li>
<p><strong>Uncertainty Baselines: Benchmarks for Uncertainty &amp; Robustness in Deep Learning.&nbsp;</strong><em>arXiv preprint arXiv:2106.04015</em>&nbsp;(2021). Zachary Nado, Neil Band, Mark Collier, Josip Djolonga, Michael W. Dusenberry, Sebastian Farquhar, Angelos Filos et al.&nbsp;[<a href="https://arxiv.org/abs/2106.04015">paper]</a></p>
</li>
<li>
<p><strong>Revisiting one-vs-all classifiers for predictive uncertainty and out-of-distribution detection in neural networks.</strong>&nbsp;<em>arXiv preprint arXiv:2007.05134</em>&nbsp;(2020). Shreyas Padhy, Zachary Nado, <em>Jie Ren,</em> Jeremiah Liu, Jasper Snoek, and Balaji Lakshminarayanan. [<a href="https://arxiv.org/abs/2007.05134">paper]</a>&nbsp;<strong> &nbsp; &nbsp;</strong></p>
</li>
</ul>
<ul>
<li class="pub-title">
<p><strong>Likelihood ratios for out-of-distribution detection. </strong><em>NeurIPS (2019)</em>.&nbsp;<em>Jie Ren</em>, Peter J. Liu, Emily Fertig, Jasper Snoek, Ryan Poplin, Mark A. DePristo, Joshua V. Dillon, and Balaji Lakshminarayanan.&nbsp;[<a href="https://arxiv.org/abs/1906.02845">paper] [</a><a href="https://github.com/google-research/google-research/tree/master/genomics_ood">code] [</a><a href="http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-llr-ood-poster.pdf">poster] [</a><a href="https://www.youtube.com/watch?v=-FduW9ZWAR4">3-minute video]&nbsp;</a>[<a href="https://ai.googleblog.com/2019/12/improving-out-of-distribution-detection.html">blog]</a></p>
</li>
<li>
<p><strong>Can you trust your model's uncertainty? Evaluating predictive uncertainty under dataset shift.</strong>&nbsp;&nbsp;<em>arXiv preprint arXiv:1906.02530</em>&nbsp;(2019). Yaniv Ovadia, Emily Fertig, <em>Jie Ren</em>, Zachary Nado, David Sculley, Sebastian Nowozin, Joshua V. Dillon, Balaji Lakshminarayanan, and Jasper Snoek.&nbsp;[<a href="https://arxiv.org/abs/1906.02530">paper</a>] [<a href="https://github.com/google-research/google-research/tree/master/uq_benchmark_2019">code</a>] [<a href="http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-benchmark-poster.pdf">poster</a>] [<a href="https://ai.googleblog.com/2020/01/can-you-trust-your-models-uncertainty.html">blog</a>]&nbsp; &nbsp; &nbsp;</p>
</li>
</ul>
<p>&nbsp;</p>
<h4>Machine Learning in Biomedical Sciences</h4>
<ul>
<li><strong>KIMI: Knockoff Inference for Motif Identification from molecular sequences with controlled false discovery rate</strong>. <em>Bioinformatics</em>&nbsp;37, no. 6 (2021): 759-766. Bai Xin, <em>Jie Ren</em>, Yingying Fan, and Fengzhu Sun. [<a href="https://academic.oup.com/bioinformatics/article-abstract/37/6/759/5942973">paper]</a>&nbsp;</li>
<li>
<p><strong>Identifying viruses from metagenomic data using deep learning.&nbsp;</strong><em>Quantitative Biology </em>(2020): 1-14<em>.&nbsp;Jie Ren*, </em>Kai Song*, Chao Deng, Nathan A. Ahlgren, Jed A. Fuhrman, Yi Li, Xiaohui Xie, Ryan Poplin, and Fengzhu Sun. [<a href="https://link.springer.com/content/pdf/10.1007/s40484-019-0187-4.pdf">paper</a>] [<a href="https://github.com/jessieren/DeepVirFinder">code</a>]</p>
</li>
</ul>
<ul>
<li class="pub-title">
<p><strong>A network-based integrated framework for predicting virus&ndash;prokaryote interactions.</strong>&nbsp;<em>NAR Genomics and Bioinformatics</em>&nbsp;2, no. 2 (2020): lqaa044. Weili Wang*, <em>Jie Ren*</em>, Kujin Tang, Emily Dart, Julio Cesar Ignacio-Espinoza, Jed A. Fuhrman, Jonathan Braun, Fengzhu Sun, and Nathan A. Ahlgren.&nbsp;[<a href="https://academic.oup.com/nargab/article/2/2/lqaa044/5861484?login=true">paper]</a>&nbsp;[<a href="https://github.com/jessieren/VirHostMatcher">code</a>]</p>
</li>
</ul>
<ul>
<li>
<p><strong>Rapid diagnosis and comprehensive bacteria profiling of sepsis based on cell-free DNA.</strong>&nbsp;<em>Journal of Translational Medicine</em>&nbsp;18, no. 1 (2020): 1-10. Pei Chen, Shuo Li, Wenyuan Li, <em>Jie Ren</em>, Fengzhu Sun, Rui Liu, and Xianghong Jasmine Zhou. [<a href="https://link.springer.com/article/10.1186/s12967-019-02186-x;">paper]</a>&nbsp;</p>
</li>
<li>
<p><strong>Afann: bias adjustment for alignment-free sequence comparison based on sequencing data using neural network regression.</strong>&nbsp;<em>Genome Biology</em> 20.1 (2019): 1-17<em>.&nbsp;</em>Kujin Tang,<em> Jie Ren</em>, and Fengzhu Sun. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1872-3">paper</a>]</p>
</li>
<li>
<p><strong>Metagenomic unmapped reads provide important insights into human microbiota and disease associations.&nbsp;</strong><em>Genome Biology</em>&nbsp;20, no. 1 (2019): 1-13. Zifan Zhu,<em> Jie Ren</em>, Sonia Michail, and Fengzhu Sun. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1773-5">paper</a>]</p>
</li>
<li><strong>Alignment-free oligonucleotide frequency dissimilarity measure improves prediction of hosts from metagenomically-derived viral sequences.&nbsp;</strong><em>Nucleic acids research </em>45.1 (2018), 39-53<em>.&nbsp;</em>Nathan A. Ahlgren*, <em>Jie Ren*</em>, Yang Y. Lu, Jed A. Fuhrman, and Fengzhu Sun. [<a href="https://academic.oup.com/nar/article/45/1/39/2605663?login=true">paper</a>] [<a href="https://github.com/jessieren/VirHostMatcher">code</a>]</li>
<li><strong>VirFinder: a novel k-mer based tool for identifying viral sequences from assembled metagenomic data.&nbsp;</strong><em>Microbiome</em>&nbsp;5, no. 1 (2017): 1-20. <em>Jie Ren*,&nbsp;</em>Nathan A. Ahlgren*, Yang Young Lu, Jed A. Fuhrman, and Fengzhu Sun.&nbsp;[<a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=Os9wmpkAAAAJ&amp;citation_for_view=Os9wmpkAAAAJ:Vno172sVVMwC">paper]</a>&nbsp;[<a href="https://github.com/jessieren/VirFinder">code</a>] [<a href="https://anaconda.org/bioconda/r-virfinder">R package</a>] [<a href="https://www.nature.com/articles/d41586-018-03358-3">news</a>]</li>
</ul>
<ul>
<li><strong>Identifying group-specific sequences for microbial communities using long k-mer sequence signatures.&nbsp;</strong><em>Frontiers in Microbiology</em>&nbsp;9 (2018): 872. Ying Wang, Lei Fu, <em>Jie Ren</em>, Zhaoxia Yu, Ting Chen, and Fengzhu Sun. [<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.00872/full">paper</a>]</li>
</ul>
<ul>
<li>
<p><strong>Prediction of virus-host infectious association by supervised learning methods.&nbsp;</strong><em>BMC Bioinformatics</em> 18.3 (2017): 143-154. Mengge Zhang, Lianping Yang, <em>Jie Ren</em>, Nathan A. Ahlgren, Jed A. Fuhrman, and Fengzhu Sun. [<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1473-7">paper</a>]</p>
<p>&nbsp;</p>
</li>
</ul>
<h4>Statistical Modeling in Genomics Data</h4>
<ul>
<li><strong>Confidence intervals for Markov chain transition probabilities based on next generation sequencing reads data.</strong>&nbsp;<em>Quantitative Biology </em>8 (2020): 143-154<em>.&nbsp;</em>Lin Wan, Xin Kang, <em>Jie Ren</em>, and Fengzhu Sun. [<a href="https://link.springer.com/content/pdf/10.1007/s40484-020-0200-y.pdf">paper</a>]&nbsp;</li>
<li class="pub-title">
<p><strong>Predicting the Number of Bases to Attain Sufficient Coverage in High-Throughput Sequencing Experiments. </strong><em>Journal of Computational Biology</em>&nbsp;27, no. 7 (2020): 1130-1143. Chao Deng, Timothy Daley, Peter Calabrese, <em>Jie Ren</em>, and Andrew D. Smith.&nbsp;[<a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2019.0264">paper]</a>&nbsp;</p>
</li>
<li>
<p><strong>Reads binning improves alignment-free metagenome comparison.</strong>&nbsp;<em>Frontiers in Genetics</em>&nbsp;10 (2019): 1156. Kai Song, <em>Jie Ren</em>, and Fengzhu Sun. [<a href="https://internal-journal.frontiersin.org/articles/10.3389/fgene.2019.01156/full">paper</a>]</p>
</li>
<li><strong>Alignment-free sequence analysis and applications.&nbsp;</strong><em>Annual Review of Biomedical Data Science</em>&nbsp;1 (2018): 93-114. <em>Jie Ren</em>, Xin Bai, Yang Young Lu, Kujin Tang, Ying Wang, Gesine Reinert, and Fengzhu Sun. [<a href="https://www.annualreviews.org/doi/abs/10.1146/annurev-biodatasci-080917-013431">paper</a>]</li>
<li>
<p><strong>Alignment-free genome comparison enables accurate geographic sourcing of white oak DNA.&nbsp;</strong><em>BMC Genomics</em>&nbsp;19, no. 1 (2018): 1-16. Kujin Tang, <em>Jie Ren</em>, Richard Cronn, David L. Erickson, Brook G. Milligan, Meaghan Parker-Forney, John L. Spouge, and Fengzhu Sun. [<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5253-1">paper</a>]</p>
</li>
<li><strong>Gut microbial and metabolomic profiles after fecal microbiota transplantation in pediatric ulcerative colitis patients</strong>.&nbsp;<em>FEMS Microbiology Ecology</em>&nbsp;94, no. 9 (2018): fiy133.&nbsp;Nusbaum, David J., Fengzhu Sun,<em> Jie Ren</em>, Zifan Zhu, Natalie Ramsy, Nicholas Pervolarakis, Sachin Kunde et al. [<a href="https://academic.oup.com/femsec/article/94/9/fiy133/5053801?login=true">paper</a>]</li>
<li>
<p><strong>Optimal choice of word length when comparing two Markov sequences using a &chi;2-statistic.&nbsp;</strong><em>BMC Genomics</em> 18.6 (2017): 19-30.&nbsp;Xin Bai, Kujin Tang, <em>Jie Ren</em>, Michael Waterman, and Fengzhu Sun. [<a href="https://link.springer.com/article/10.1186/s12864-017-4020-z">paper</a>]</p>
</li>
<li>
<p><strong>CAFE: aCcelerated&nbsp;Alignment-FrEe sequence analysis.&nbsp;</strong><em>Nucleic Acids Research</em> 45.1 (2017): W554-W559.&nbsp;Yang Y. Lu, Kujin Tang, Jie Ren, Jed A. Fuhrman, Michael S. Waterman, and Fengzhu Sun. [<a href="https://academic.oup.com/nar/article/45/W1/W554/3791208?login=true">paper</a>]&nbsp;</p>
</li>
<li><strong>Alignment-free transcriptomic and metatranscriptomic comparison using sequencing signatures with variable length markov chains.&nbsp;</strong><em>Scientific Reports</em> 6.1 (2016): 1-15. Weinan Liao*, <em>Jie Ren*</em>, Kun Wang, Shun Wang, Feng Zeng, Ying Wang, and Fengzhu Sun. [<a href="https://www.nature.com/articles/srep37243">paper</a>]</li>
<li>
<p><strong>Inference of Markovian properties of molecular sequences from NGS data and applications to comparative genomics.&nbsp;</strong><em>Bioinformatics </em>32.7 (2016): 993-1000.&nbsp;<em>Jie Ren</em>, Kai Song, Minghua Deng, Gesine Reinert, Charles H. Cannon, and Fengzhu Sun. [<a href="https://academic.oup.com/bioinformatics/article/32/7/993/2240327?login=true">paper</a>]</p>
</li>
<li><strong>New developments of alignment-free sequence comparison: measures, statistics and next-generation sequencing.&nbsp;</strong><em>Briefings in Bioinformatics</em> 15.3 (2014): 343-353.&nbsp;Kai Song, <em>Jie Ren</em>, Gesine Reinert, Minghua Deng, Michael S. Waterman, and Fengzhu Sun. [<a href="https://academic.oup.com/bib/article/15/3/343/182355?login=true">paper</a>]</li>
<li>
<p><strong>Multiple alignment-free sequence comparison.&nbsp;</strong><em>Bioinformatics 29.21 (2013): 2690-2698.&nbsp;Jie Ren</em>, Kai Song, Fengzhu Sun, Minghua Deng, and Gesine Reinert. [<a href="https://academic.oup.com/bioinformatics/article/29/21/2690/195361?login=true">paper</a>]</p>
</li>
<li><strong>Alignment-free sequence comparison based on next-generation sequencing reads.&nbsp;</strong><em>Journal of Computational Biology</em> 20.2 (2013): 64-79. Kai Song, <em>Jie Ren,</em> Zhiyuan Zhai, Xuemei Liu, Minghua Deng, and Fengzhu Sun. [<a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2012.0228">paper</a>]</li>
<li>
<p><strong>Comparison of metagenomic samples using sequence signatures.&nbsp;</strong>BMC Genomics 13.1 (2012): 1-17.<strong>&nbsp;</strong>Bai Jiang*, Kai Song*, <em>Jie Ren*</em>, Minghua Deng, Fengzhu Sun, and Xuegong Zhang.</p>
</li>
</ul>
<p>&nbsp;</p>
<h4>Natural Language Processing</h4>
<ul>
<li><strong>SummAE: Zero-shot abstractive text summarization using length-agnostic auto-encoders.</strong>&nbsp;<em>arXiv preprint arXiv:1910.00998.</em> Peter J. Liu, Yu-An Chung, and&nbsp;<em>Jie Ren. [<a href="https://arxiv.org/abs/1910.00998">paper</a>]</em></li>
</ul>
<p>&nbsp;</p>
<h2>Talks</h2>
<ul class="pub">
<li>2021 ICML WiML Un-Workshop:&nbsp;<a href="https://docs.google.com/presentation/d/1E90g5yJpYZBa-_jRYGc0IMSnE0axf0wHskUKS1XD-9Y/edit?usp=sharing">Uncertainty estimation and out-of-distribution (OOD) detection in deep learning.</a></li>
<li>2019 NeurIPS:&nbsp;<a href="https://www.youtube.com/watch?v=-FduW9ZWAR4">Likelihood ratios for out-of-distribution detection.</a></li>
<li>2016 Molecular and Computational Biology Retreat, University of Southern California: Predicting virus-host interactions using sequence signatures.</li>
<li>2015 Joint USC/UCLA Bioinformatics Colloquium: Statistical inference of Markov models of genomic sequences using NGS data.</li>
<li>2015 Research in Computational Molecular Biology (RECOMB)-Seq: Inference of Markovian properties of molecular sequences from NGS data and applications to comparative genomics.</li>
<li>2013 Molecular and Computational Biology Retreat, University of Southern California: Alignment-free Genome and Metagenome Comparison Based On Next Generation Sequencing Reads.</li>
</ul>
<p>&nbsp;</p>
<h2>Academic Activities</h2>
<ul>
<li>Reviewer for machine learning conferences: NeurIPS (2019-), ICML (2019-), ICLR (2019-)</li>
<li>Program Committee (PC) for ICML workshop on Unvertainty and Robustness in Deep Learning&nbsp; (2020-2021)&nbsp;</li>
<li>Reviewer for the following Bioinformatics journals: Cell Systems, Nucleic Acids Research, Gut, Bioinformatics, Giga Science, Scientific Reports, PLOS Computational Biolofy, PLOS ONE.&nbsp;</li>
</ul>
<p>&nbsp;</p>
<h2>Education</h2>
<ul>
<li>Postdoc Fellow, University of Southern Califonia, Los Angeles, USA. (2017-2018).</li>
<li>Ph.D. in Computational Biology and Bioinformatics, University of Southern California, Los Angeles, USA. (2013-2017).
<ul>
<li>Advisors: <a href="https://dornsife.usc.edu/cf/faculty-and-staff/faculty.cfm?pid=1003742">Fengzhu Sun</a>, <a href="https://dornsife.usc.edu/cf/faculty-and-staff/faculty.cfm?pid=1003269">Jed Fuhrman</a>, <a href="https://dornsife.usc.edu/labs/msw/">Michael Waterman</a>, <a href="http://www.stats.ox.ac.uk/~reinert/">Gesine Reinert</a></li>
</ul>
</li>
<li>M.S. in Statistics, University of Southern California, Los Angeles, USA. (2016-2017).</li>
<li>M.S. in Probability and Statistics, Peking University, Beijing, China. (2010-2013).</li>
<li>B.S. in Information and Computing Science, Minzu University, Beijing, China. (2006-2010).</li>
</ul>
<p>&nbsp;</p>
</div>

<!-- attempt hiding from spambots :p -->
<script type="text/javascript">
var e_is_shown = false;
document.getElementById('iemail').addEventListener("click", function(){
    let demail = document.getElementById('demail');
    demail.innerHTML = 'renj' + ' _at_ ' + 'usc' + '.edu';
    demail.style.opacity = e_is_shown ? 0 : 1;
    e_is_shown = !e_is_shown;
})
</script>

</body>
</html>
