<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <title></title>
  <meta name="Generator" content="Cocoa HTML Writer">
  <meta name="CocoaVersion" content="2299.3">
  <style type="text/css">
    p.p1 {margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c}
    p.p2 {margin: 0.0px 0.0px 0.0px 0.0px; font: 32.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c}
    p.p3 {margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Times; color: #0000e3; -webkit-text-stroke: #0000e3}
    p.p4 {margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Times; color: #333333; -webkit-text-stroke: #333333}
    p.p5 {margin: 0.0px 0.0px 19.9px 0.0px; font: 24.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c}
    p.p6 {margin: 0.0px 0.0px 16.0px 0.0px; font: 16.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c}
    p.p7 {margin: 0.0px 0.0px 21.2px 0.0px; font: 16.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c}
    p.p8 {margin: 0.0px 0.0px 21.2px 0.0px; font: 16.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c; min-height: 19.0px}
    p.p9 {margin: 0.0px 0.0px 16.0px 0.0px; font: 16.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c; min-height: 19.0px}
    li.li1 {margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c}
    li.li6 {margin: 0.0px 0.0px 16.0px 0.0px; font: 16.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c}
    li.li7 {margin: 0.0px 0.0px 21.2px 0.0px; font: 16.0px Times; color: #0c0c0c; -webkit-text-stroke: #0c0c0c}
    li.li10 {margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Times; color: #0000e3; -webkit-text-stroke: #0c0c0c}
    span.s1 {font-kerning: none}
    span.s2 {text-decoration: underline ; font-kerning: none; color: #0000e9}
    span.s3 {font-kerning: none; color: #0c0c0c; -webkit-text-stroke: 0px #0c0c0c}
    span.s4 {text-decoration: underline ; font-kerning: none; color: #0000e3; -webkit-text-stroke: 0px #0000e3}
    span.s5 {text-decoration: underline ; font-kerning: none; color: #0c0c0c; -webkit-text-stroke: 0px #0c0c0c}
    span.s6 {text-decoration: underline ; font-kerning: none; color: #0000e3}
    span.s7 {color: #0c0c0c}
    span.s8 {font-kerning: none; color: #0c0c0c}
    ul.ul1 {list-style-type: disc}
    ul.ul2 {list-style-type: circle}
  </style>
</head>
<body>

<div id="wrap">
<div id="links">&nbsp;</div>
<h1>Jie Ren</h1>
<h1>&nbsp;<img src="https://scholar.googleusercontent.com/citations?view_op=medium_photo&amp;user=Os9wmpkAAAAJ&amp;citpid=10" alt="" width="120" height="120" /></h1>
<div>Computational Biology and Bioinformatics; Machine learning; Statistical Modeling</div>
<div id="links"><a href="https://twitter.com/jessierenjie">Twitter</a> <a href="https://github.com/jessieren">Github</a> <a href="https://scholar.google.com/citations?user=Os9wmpkAAAAJ&amp;hl=en">Google Scholar</a> <span id="iemail" style="cursor: pointer; text-decoration: underline;">Email</span></div>
<div id="demail">&nbsp;</div>
<!-- will reveal email -->
<div>I am a Senior Research Scientist in <a href="https://research.google/teams/brain/">Google Research Brain Team</a>. Previously, I received my PhD in <a href="https://www.qcb-dornsife.usc.edu/">Computational Biology and Bioinformatics</a> and MSc in Statistics at the University of Southern California. I am interested in <span style="font-weight: 400;">developing robust and reliable methods for biomedical data analysis, with a focus on uncertainty quantification and out-of-distribution (OOD) detection in deep learning.</span></div>
<div>&nbsp;</div>
<p class="p5"><span class="s1"><b>News</b></span></p>
<ul class="ul1">
  <li class="li1"><span class="s1">12/2023: Our study on selective generation for large language models, "Self-evaluation improves selective generation in large language models”, will be presented (spotlight) at NeurIPS 2023 I Can’t Believe It’s Not Better! (ICBINB) Workshop [<a href="https://openreview.net/forum?id=OptKBWmreP"><span class="s2">paper</span></a>].</span></li>
  <li class="li1"><span class="s1">07/2023: Our work on prompt selection and prompt ensemble for multimodal models, "A simple zero-shot prompt weighting technique to improve prompt ensembling in text-image models", will be presented at ICML 2023. [<a href="https://proceedings.mlr.press/v202/allingham23a.html"><span class="s6">paper]</span></a> [<a href="https://twitter.com/JamesAllingham/status/1684125147756888064"><span class="s6">twitter]</span></a> [<a href="https://github.com/google/uncertainty-baselines/tree/main/experimental/multimodal"><span class="s6">code]</span></a> .</span></li>
  <li class="li1"><span class="s1">06/2023: Our paper "Improving Zero-shot Generalization and Robustness of Multi-modal Models" is accepted by CVPR 2023. Congratulations to Yunhao Ge. [<a href="https://arxiv.org/pdf/2212.01758.pdf"><span class="s6">paper]</span></a> [<a href="https://twitter.com/GeYunhao/status/1664478907012366337"><span class="s6">twitter]</span></a> [<a href="https://github.com/gyhandy/Hierarchy-CLIP"><span class="s6">code]</span></a> .</span></li>
  <li class="li1"><span class="s1">05/2023: Our paper "Out-of-Distribution Detection and Selective Generation for Conditional Language Models" is accepted (spotlight) by ICLR 2023. [<a href="https://arxiv.org/abs/2209.15558"><span class="s6">paper]</span></a> [<a href="https://twitter.com/peterjliu/status/1620157580114030592"><span class="s6">twitter]</span></a> .</span></li>
  <li class="li1"><span class="s1">07/2022: Together with my colleagues at Google, we release a framework, Plex, for reliability in large pre-trained models. Please check out our latest blog post to learn more. [<a href="https://ai.googleblog.com/2022/07/towards-reliability-in-deep-learning.html"><span class="s6">blog]</span></a> .</span></li>
  <li class="li1"><span class="s1">01/2022: Our new AI blog post is out :) It walks through our recent paper on enabling dermatology AI systems to detect the presence of conditinos unseen in training [<a href="https://ai.googleblog.com/2022/01/does-your-medical-image-classifier-know.html"><span class="s6">blog]</span></a> .</span></li>
  <li class="li1"><span class="s1">12/2021: Our paper "Exploring the Limits of Out-of-Distribution Detection" [<a href="https://arxiv.org/abs/2106.03004"><span class="s6">paper]</span></a> will be presented at NeurIPS 2021.</span></li>
  <li class="li1"><span class="s1">07/2021: Together with <a href="https://polkirichenko.github.io/"><span class="s6">Polina Kirichenko</span></a>, <a href="https://scholar.google.com/citations?user=-SjNMu8AAAAJ&amp;hl=en"><span class="s6">Haleh Akrami</span></a>, <a href="http://pages.cs.wisc.edu/~sharonli/"><span class="s6">Sharon Yixuan Li</span></a> and <a href="https://scholar.google.com/citations?user=D_CuHyUAAAAJ&amp;hl=en"><span class="s6">Sergül Aydöre</span></a>, we are going to lead a <a href="https://icml.cc/virtual/2021/affinityworkshop/10849"><span class="s6">breakout session</span></a> at July 21 7:25 pm ET at WiML Un-Workshop at ICML 2021 on "Does your model know what it doesn’t know? Uncertainty estimation and OOD detection in deep learning."</span></li>
  <li class="li1"><span class="s1">07/2021: Our paper "A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection" [<a href="https://arxiv.org/abs/2106.09022"><span class="s6">paper]</span></a> [<a href="https://twitter.com/jessierenjie/status/1418321220546613248"><span class="s6">poster]</span></a> will be presented at ICML 2021 UDL workshop.</span></li>
  <li class="li1"><span class="s1">07/2021: Our paper "Exploring the Limits of Out-of-Distribution Detection" [<a href="https://arxiv.org/abs/2106.03004"><span class="s6">paper]</span></a> will be presented at ICML 2021 UDL workshop.</span></li>
  <li class="li1"><span class="s1">12/2019: We will present our work of "Likelihood Ratios for Out-of-Distribution Detection" at NeurIPS 2019. [<a href="https://arxiv.org/abs/1906.02845"><span class="s6">paper</span></a>][<a href="https://arxiv.org/abs/1906.02845"><span class="s6">paper] [</span></a><a href="https://github.com/google-research/google-research/tree/master/genomics_ood"><span class="s6">code] [</span></a><a href="http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-llr-ood-poster.pdf"><span class="s6">poster] [</span></a><a href="https://www.youtube.com/watch?v=-FduW9ZWAR4"><span class="s6">3-minute video] </span></a>[<a href="https://ai.googleblog.com/2019/12/improving-out-of-distribution-detection.html"><span class="s6">blog]</span></a></span></li>
  <li class="li1"><span class="s1">06/2019: I am joining Google Brain as a Research Scientist. </span></li>
  <li class="li1"><span class="s1">06/2018: I am joining Google Brain AI Residency program. </span></li>
  <li class="li1"><span class="s1">03/2018: Our work is highlighted in <a href="https://www.nature.com/articles/d41586-018-03358-3"><span class="s6"><i>Nature News </i></span></a>titled "Machine learning spots treasure trove of elusive viruses Artificial intelligence could speed up metagenomic studies that look for species unknown to science.".</span></li>
  <li class="li1"><span class="s1">04/2017: I am honored to receive the prestigious <a href="https://twitter.com/uscbiosci/status/858070439435710464"><span class="s6">USC PhD Achievement Award</span></a> (6 winners university-wide per year).</span></li>
  <li class="li1"><span class="s1">03/2017: I am honored to receive Women in Science and Engineering Travel Grant. </span></li>
  <li class="li1"><span class="s1">04/2016: I am honored to receive <a href="https://dornsife.usc.edu/cams/cams-prize/"><span class="s6">USC CAMS Graduate Student Prize</span></a> for Excellence in Research with a Substantial Mathematical Component (2 winners university-wide per year). </span></li>
</ul>
<p class="p6"><span class="s1"> </span></p>
<p class="p5"><span class="s1"><b>Publications</b></span></p>
<p class="p7"><span class="s1"><b>Out-of-Distribution Detection in Deep Learning</b></span></p>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Building One-class Detector for Anything: Open-vocabulary Zero-shot OOD Detection Using Text-image Models. </b>Presented at the <i>ICML workshop on Knowledge and Logical Reasoning in the Era of Data-driven Learning<span class="Apple-converted-space">  </span></i>(2023).  Yunhao Ge*, <i>Jie Ren*</i>, Jiaping Zhao, Kaifeng Chen, Andrew Gallagher, Laurent Itti, Balaji Lakshminarayanan.[<a href="https://arxiv.org/abs/2305.17207"><span class="s6">paper]</span></a>   </span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Out-of-Distribution Detection and Selective Generation for Conditional Language Models. </b><i>ICLR</i> (2023).  <i>Jie Ren</i>, Jiaming Luo, Yao Zhao, Kundan Krishna, Mohammad Saleh, Balaji Lakshminarayanan, Peter J. Liu.[<a href="https://arxiv.org/abs/2209.15558"><span class="s6">paper]</span></a>   </span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>A Simple Approach to Improve Single-Model Deep Uncertainty via Distance-Awareness. </b><i>Journal of Machine Learning Research</i> 23 (2022): 1-63.  Jeremiah Zhe Liu*, Shreyas Padhy*, <i>Jie Ren*</i>, Zi Lin, Yeming Wen, Ghassen Jerfel, Zack Nado, Jasper Snoek, Dustin Tran, and Balaji Lakshminarayanan.[<a href="https://www.jmlr.org/papers/volume24/22-0479/22-0479.pdf"><span class="s6">paper]</span></a>   </span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>MLR-OOD: A Markov Chain Based Likelihood Ratio Method for Out-Of-Distribution Detection of Genomic Sequences. </b><i>Journal of Molecular Biology</i> (2022).  Xin Bai, <i>Jie Ren</i>, Fengzhu Sun.[<a href="https://www.sciencedirect.com/science/article/pii/S0022283622001668"><span class="s6">paper]</span></a>[<a href="https://docs.google.com/presentation/d/1XmsWPViWmXh0cwOBFRXlBE2CnsWyaynzN9Qrmv2pNRA/edit?usp=sharing"><span class="s6">slides]</span></a>  </span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Exploring the Limits of Out-of-Distribution Detection. </b><i>NeurIPS</i> (2021).  Stanislav Fort*, <i>Jie Ren*</i>, and Balaji Lakshminarayanan.[<a href="https://arxiv.org/abs/2106.03004"><span class="s6">paper]</span></a>   </span></li>
  <li class="li6"><b></b><span class="s1"><b>A Simple Fix to Mahalanobis Distance for Improving Near-OOD Detection.  </b>Presented at the <i>ICML workshop on Uncertainty and Robustness in Deep Learning</i> (2021).  <i>Jie Ren</i>, Stanislav Fort, Jeremiah Liu, Abhijit Guha Roy, Shreyas Padhy, and Balaji Lakshminarayanan. [<a href="https://arxiv.org/abs/2106.09022"><span class="s6">paper] </span></a><a href="https://twitter.com/jessierenjie/status/1418321220546613248"><span class="s6">[poster]</span></a></span><span class="s6"> [<a href="https://github.com/google/uncertainty-baselines/blob/main/experimental/near_ood/vit/ood_utils.py"><span class="s2">code</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Does Your Dermatology Classifier Know What It Doesn't Know? Detecting the Long-Tail of Unseen Conditions.</b> <i>Medical Image Analysis</i> 75 (2022): 102274. Abhijit Guha Roy*, <i>Jie Ren*</i>, Shekoofeh Azizi, Aaron Loh, Vivek Natarajan, Basil Mustafa, Nick Pawlowski et al. [<a href="https://arxiv.org/abs/2104.03829"><span class="s6">paper]</span></a> [<a href="https://ai.googleblog.com/2022/01/does-your-medical-image-classifier-know.html"><span class="s6">blog</span></a>]</span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Revisiting One-vs-All Classifiers for Predictive Uncertainty and Out-of-Distribution Detection in Neural Networks.  </b>Presented at the <i>ICML workshop on Uncertainty and Robustness in Deep Learning</i> (2020).  Shreyas Padhy, Zachary Nado, <i>Jie Ren,</i> Jeremiah Liu, Jasper Snoek, and Balaji Lakshminarayanan. [<a href="https://arxiv.org/abs/2007.05134"><span class="s6">paper]</span></a> <b>    </b></span></li>
  <li class="li6"><b></b><span class="s1"><b>Likelihood Ratios for Out-of-Distribution Detection. </b><i>NeurIPS</i> (2019). <i>Jie Ren</i>, Peter J. Liu, Emily Fertig, Jasper Snoek, Ryan Poplin, Mark A. DePristo, Joshua V. Dillon, and Balaji Lakshminarayanan. [<a href="https://arxiv.org/abs/1906.02845"><span class="s6">paper] [</span></a><a href="https://github.com/google-research/google-research/tree/master/genomics_ood"><span class="s6">code] [</span></a><a href="http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-llr-ood-poster.pdf"><span class="s6">poster] [</span></a><a href="https://www.youtube.com/watch?v=-FduW9ZWAR4"><span class="s6">3-minute video] </span></a>[<a href="https://ai.googleblog.com/2019/12/improving-out-of-distribution-detection.html"><span class="s6">blog]</span></a>  </span></li>
</ul>
<p class="p8"><span class="s1"></span><br></p>
<p class="p7"><span class="s1"><b>Uncertainty and Robustness in Deep Learning</b></span></p>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Universal Self-Consistency for Large Language Model Generation.</b> arXiv preprint arXiv:2311.17311 (2023). Chen, Xinyun, Renat Aksitov, Uri Alon, <i>Jie Ren</i>, Kefan Xiao, Pengcheng Yin, Sushant Prakash, Charles Sutton, Xuezhi Wang, and Denny Zhou.<span class="Apple-converted-space"> </span></span></li>
  <li class="li6"><b></b><span class="s1"><b>On Uncertainty Calibration and Selective Generation in Probabilistic Neural Summarization: A Benchmark Study. </b><i>EMNLP</i> (2023). Polina Zablotskaia, Du Phan, Joshua Maynez, Shashi Narayan, <i>Jie Ren</i>, Jeremiah Liu. [<a href="https://arxiv.org/abs/2304.08653"><span class="s2">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Improving the Robustness of Summarization Models by Detecting and Removing Input Noise. </b><i>EMNLP</i> (2023).  Kundan Krishna, Yao Zhao, <i>Jie Ren</i>, Balaji Lakshminarayanan, Jiaming Luo, Mohammad Saleh, Peter J. Liu.[<a href="https://arxiv.org/abs/2212.09928"><span class="s6">paper]</span></a>   </span></li>
  <li class="li6"><b></b><span class="s1"><b>A Simple Zero-shot Prompt Weighting Technique to Improve Prompt Ensembling in Text-Image Models. </b><i>ICML</i> (2023).  James Urquhart Allingham*, <i>Jie Ren*</i>, Michael W Dusenberry, Jeremiah Zhe Liu, Xiuye Gu, Yin Cui, Dustin Tran, Balaji Lakshminarayanan.[<a href="https://arxiv.org/abs/2302.06235"><span class="s6">paper]</span></a> <span class="Apple-converted-space"> </span></span></li>
  <li class="li6"><b></b><span class="s1"><b>Morse Neural Networks for Uncertainty Quantification.</b> Presented at<i> the ICML workshop on</i> <i>Structured Probabilistic Inference and Generative Modeling </i>Benoit Dherin, Huiyi Hu, <i>Jie Ren</i>, Michael W. Dusenberry, and Balaji Lakshminarayanan.[<a href="https://arxiv.org/abs/2307.00667"><span class="s2">paper</span></a>] </span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Improving Zero-shot Generalization and Robustness of Multi-modal Models. </b><i>CVPR</i> (2022).  Yunhao Ge*, <i>Jie Ren*</i>, Yuxiao Wang, Andrew Gallagher, Ming-Hsuan Yang, Laurent Itti, Hartwig Adam, Balaji Lakshminarayanan, Jiaping Zhao.[<a href="https://arxiv.org/abs/2212.01758"><span class="s6">paper]</span></a>   </span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Plex: Towards reliability using pretrained large model extensions. </b><i>arXiv preprint arXiv:2207.07411</i> (2022).  Dustin Tran, Jeremiah Liu, Michael W Dusenberry, Du Phan, Mark Collier, <i>Jie Ren</i>, Kehang Han, Zi Wang, Zelda Mariet, Huiyi Hu, Neil Band, Tim GJ Rudner, Karan Singhal, Zachary Nado, Joost van Amersfoort, Andreas Kirsch, Rodolphe Jenatton, Nithum Thain, Honglin Yuan, Kelly Buchanan, Kevin Murphy, D Sculley, Yarin Gal, Zoubin Ghahramani, Jasper Snoek, Balaji Lakshminarayanan.[<a href="https://arxiv.org/abs/2207.07411"><span class="s6">paper]</span></a>   </span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Uncertainty Baselines: Benchmarks for Uncertainty &amp; Robustness in Deep Learning. </b>Presented at the <i>NeurIPS workshop on Bayesian deep learning</i> (2021). Zachary Nado, Neil Band, Mark Collier, Josip Djolonga, Michael W. Dusenberry, Sebastian Farquhar, Angelos Filos et al. [<a href="https://arxiv.org/abs/2106.04015"><span class="s6">paper]</span></a></span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>SummAE: Summae: Zero-Shot Abstractive Text Summarization Using Length-Agnostic Auto-Encoders.</b> <i>arXiv preprint arXiv:1910.00998.</i> Peter J. Liu, Yu-An Chung, and <i>Jie Ren. [</i><a href="https://arxiv.org/abs/1910.00998"><span class="s6"><i>paper</i></span></a><i>]</i></span></li>
  <li class="li6"><b></b><span class="s1"><b>Can You Trust Your Model’s Uncertainty? Evaluating Predictive Uncertainty Under Dataset Shift. </b><i>NeurIPS</i> (2019).  Yaniv Ovadia, Emily Fertig, <i>Jie Ren</i>, Zachary Nado, David Sculley, Sebastian Nowozin, Joshua V. Dillon, Balaji Lakshminarayanan, and Jasper Snoek. [<a href="https://arxiv.org/abs/1906.02530"><span class="s6">paper</span></a>] [<a href="https://github.com/google-research/google-research/tree/master/uq_benchmark_2019"><span class="s6">code</span></a>] [<a href="http://www.gatsby.ucl.ac.uk/~balaji/NeurIPS2019-benchmark-poster.pdf"><span class="s6">poster</span></a>] [<a href="https://ai.googleblog.com/2020/01/can-you-trust-your-models-uncertainty.html"><span class="s6">blog</span></a>]     </span></li>
</ul>
<p class="p6"><span class="s1"> </span></p>
<p class="p7"><span class="s1"><b>Machine Learning in Biological Sciences</b></span></p>
<ul class="ul1">
  <li class="li7"><b></b><span class="s1"><b>KIMI: Knockoff Inference for Motif Identification From Molecular Sequences With Controlled False Discovery Rate</b>. <i>Bioinformatics</i> 37, no. 6 (2021): 759-766. Bai Xin, <i>Jie Ren</i>, Yingying Fan, and Fengzhu Sun. [<a href="https://academic.oup.com/bioinformatics/article-abstract/37/6/759/5942973"><span class="s6">paper]</span></a> </span></li>
  <li class="li7"><b></b><span class="s1"><b>Identifying Viruses From Metagenomic Data Using Deep Learning. </b><i>Quantitative Biology </i>(2020): 1-14<i>. Jie Ren*, </i>Kai Song*, Chao Deng, Nathan A. Ahlgren, Jed A. Fuhrman, Yi Li, Xiaohui Xie, Ryan Poplin, and Fengzhu Sun. [<a href="https://link.springer.com/content/pdf/10.1007/s40484-019-0187-4.pdf"><span class="s6">paper</span></a>] [<a href="https://github.com/jessieren/DeepVirFinder"><span class="s6">code</span></a>]</span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>A Network-Based Integrated Framework for Predicting Virus–Prokaryote Interactions.</b> <i>NAR Genomics and Bioinformatics</i> 2, no. 2 (2020): lqaa044. Weili Wang*, <i>Jie Ren*</i>, Kujin Tang, Emily Dart, Julio Cesar Ignacio-Espinoza, Jed A. Fuhrman, Jonathan Braun, Fengzhu Sun, and Nathan A. Ahlgren. [<a href="https://academic.oup.com/nargab/article/2/2/lqaa044/5861484?login=true"><span class="s6">paper]</span></a> [<a href="https://github.com/jessieren/VirHostMatcher"><span class="s6">code</span></a>]</span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Rapid Diagnosis and Comprehensive Bacteria Profiling of Sepsis Based on Cell-Free Dna. </b><i>Journal of Translational Medicine</i> 18, no. 1 (2020): 1-10. Pei Chen, Shuo Li, Wenyuan Li, <i>Jie Ren</i>, Fengzhu Sun, Rui Liu, and Xianghong Jasmine Zhou. [<a href="https://link.springer.com/article/10.1186/s12967-019-02186-x;"><span class="s6">paper]</span></a> </span></li>
  <li class="li6"><b></b><span class="s1"><b>Afann: Bias Adjustment for Alignment-Free Sequence Comparison Based on Sequencing Data Using Neural Network Regression.</b> <i>Genome Biology</i> 20.1 (2019): 1-17<i>. </i>Kujin Tang,<i> Jie Ren</i>, and Fengzhu Sun. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1872-3"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Metagenomic Unmapped Reads Provide Important Insights Into Human Microbiota and Disease Associations. </b><i>Genome Biology</i> 20, no. 1 (2019): 1-13. Zifan Zhu,<i> Jie Ren</i>, Sonia Michail, and Fengzhu Sun. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1773-5"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Alignment-Free Oligonucleotide Frequency Dissimilarity Measure Improves Prediction of Hosts From Metagenomically-Derived Viral Sequences. </b><i>Nucleic Acids Research </i>45.1 (2018), 39-53<i>. </i>Nathan A. Ahlgren*, <i>Jie Ren*</i>, Yang Y. Lu, Jed A. Fuhrman, and Fengzhu Sun. [<a href="https://academic.oup.com/nar/article/45/1/39/2605663?login=true"><span class="s6">paper</span></a>] [<a href="https://github.com/jessieren/VirHostMatcher"><span class="s6">code</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>VirFinder: A Novel K-Mer Based Tool for Identifying Viral Sequences From Assembled Metagenomic Data. </b><i>Microbiome</i> 5, no. 1 (2017): 1-20. <i>Jie Ren*, </i>Nathan A. Ahlgren*, Yang Young Lu, Jed A. Fuhrman, and Fengzhu Sun. [<a href="https://link.springer.com/article/10.1186/s40168-017-0283-5"><span class="s6">paper]</span></a> [<a href="https://github.com/jessieren/VirFinder"><span class="s6">code</span></a>] [<a href="https://anaconda.org/bioconda/r-virfinder"><span class="s6">R package</span></a>] [<a href="https://www.nature.com/articles/d41586-018-03358-3"><span class="s6">news</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Identifying Group-Specific Sequences for Microbial Communities Using Long K-Mer Sequence Signatures. </b><i>Frontiers in Microbiology</i> 9 (2018): 872. Ying Wang, Lei Fu, <i>Jie Ren</i>, Zhaoxia Yu, Ting Chen, and Fengzhu Sun. [<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.00872/full"><span class="s6">paper</span></a>]</span></li>
  <li class="li1"><b></b><span class="s1"><b>Prediction of Virus-Host Infectious Association by Supervised Learning Methods. </b><i>BMC Bioinformatics</i> 18.3 (2017): 143-154. Mengge Zhang, Lianping Yang, <i>Jie Ren</i>, Nathan A. Ahlgren, Jed A. Fuhrman, and Fengzhu Sun. [<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1473-7"><span class="s6">paper</span></a>]</span></li>
</ul>
<p class="p9"><span class="s1"></span><br></p>
<p class="p7"><span class="s1"><b>Statistical Modeling in Genomics Data</b></span></p>
<ul class="ul1">
  <li class="li7"><b></b><span class="s1"><b>A New Context Tree Inference Algorithm for Variable Length Markov Chain Model with Applications to Biological Sequence Analyses.</b> <i>Journal of Computational Biology</i> (2022). Shaokun An, <i>Jie Ren</i>, Fengzhu Sun, and Lin Wan. [<a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0604"><span class="s6">paper</span></a>]</span></li>
</ul>
<ul class="ul1">
  <li class="li6"><b></b><span class="s1"><b>Confidence Intervals for Markov Chain Transition Probabilities Based on Next Generation Sequencing Reads Data.</b> <i>Quantitative Biology </i>8 (2020): 143-154<i>. </i>Lin Wan, Xin Kang, <i>Jie Ren</i>, and Fengzhu Sun. [<a href="https://link.springer.com/article/10.1007/s40484-020-0200-y"><span class="s6">paper</span></a>] </span></li>
  <li class="li6"><b></b><span class="s1"><b>Predicting the Number of Bases to Attain Sufficient Coverage in High-Throughput Sequencing Experiments. </b><i>Journal of Computational Biology</i> 27, no. 7 (2020): 1130-1143. Chao Deng, Timothy Daley, Peter Calabrese, <i>Jie Ren</i>, and Andrew D. Smith. [<a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2019.0264"><span class="s6">paper]</span></a> </span></li>
  <li class="li6"><b></b><span class="s1"><b>Reads Binning Improves Alignment-Free Metagenome Comparison.</b> <i>Frontiers in Genetics</i> 10 (2019): 1156. Kai Song, <i>Jie Ren</i>, and Fengzhu Sun. [<a href="https://internal-journal.frontiersin.org/articles/10.3389/fgene.2019.01156/full"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Alignment-Free Sequence Analysis and Applications. </b><i>Annual Review of Biomedical Data Science</i> 1 (2018): 93-114. <i>Jie Ren</i>, Xin Bai, Yang Young Lu, Kujin Tang, Ying Wang, Gesine Reinert, and Fengzhu Sun. [<a href="https://www.annualreviews.org/doi/abs/10.1146/annurev-biodatasci-080917-013431"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Alignment-Free Genome Comparison Enables Accurate Geographic Sourcing of White Oak DNA. </b><i>BMC Genomics</i> 19, no. 1 (2018): 1-16. Kujin Tang, <i>Jie Ren</i>, Richard Cronn, David L. Erickson, Brook G. Milligan, Meaghan Parker-Forney, John L. Spouge, and Fengzhu Sun. [<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5253-1"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Gut Microbial and Metabolomic Profiles After Fecal Microbiota Transplantation in Pediatric Ulcerative Colitis Patients.</b> <i>FEMS Microbiology Ecology</i> 94, no. 9 (2018): fiy133. Nusbaum, David J., Fengzhu Sun,<i> Jie Ren</i>, Zifan Zhu, Natalie Ramsy, Nicholas Pervolarakis, Sachin Kunde et al. [<a href="https://academic.oup.com/femsec/article/94/9/fiy133/5053801?login=true"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Optimal Choice of Word Length When Comparing Two Markov Sequences Using a χ2-Statistic. </b><i>BMC Genomics</i> 18.6 (2017): 19-30. Xin Bai, Kujin Tang, <i>Jie Ren</i>, Michael Waterman, and Fengzhu Sun. [<a href="https://link.springer.com/article/10.1186/s12864-017-4020-z"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Cafe: Accelerated Alignment-Free Sequence Analysis. </b><i>Nucleic Acids Research</i> 45.1 (2017): W554-W559. Yang Y. Lu, Kujin Tang, Jie Ren, Jed A. Fuhrman, Michael S. Waterman, and Fengzhu Sun. [<a href="https://academic.oup.com/nar/article/45/W1/W554/3791208?login=true"><span class="s6">paper</span></a>] </span></li>
  <li class="li6"><b></b><span class="s1"><b>Alignment-Free Transcriptomic and Metatranscriptomic Comparison Using Sequencing Signatures With Variable Length Markov Chains. </b><i>Scientific Reports</i> 6.1 (2016): 1-15. Weinan Liao*, <i>Jie Ren*</i>, Kun Wang, Shun Wang, Feng Zeng, Ying Wang, and Fengzhu Sun. [<a href="https://www.nature.com/articles/srep37243"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Inference of Markovian Properties of Molecular Sequences From Ngs Data and Applications to Comparative Genomics. </b><i>Bioinformatics </i>32.7 (2016): 993-1000. <i>Jie Ren</i>, Kai Song, Minghua Deng, Gesine Reinert, Charles H. Cannon, and Fengzhu Sun. [<a href="https://academic.oup.com/bioinformatics/article/32/7/993/2240327?login=true"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>New Developments of Alignment-Free Sequence Comparison: Measures, Statistics and Next-Generation Sequencing. </b><i>Briefings in Bioinformatics</i> 15.3 (2014): 343-353. Kai Song, <i>Jie Ren</i>, Gesine Reinert, Minghua Deng, Michael S. Waterman, and Fengzhu Sun. [<a href="https://academic.oup.com/bib/article/15/3/343/182355?login=true"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Multiple Alignment-Free Sequence Comparison. </b><i>Bioinformatics 29.21 (2013): 2690-2698. Jie Ren</i>, Kai Song, Fengzhu Sun, Minghua Deng, and Gesine Reinert. [<a href="https://academic.oup.com/bioinformatics/article/29/21/2690/195361?login=true"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Alignment-Free Sequence Comparison Based on Next-Generation Sequencing Reads. </b><i>Journal of Computational Biology</i> 20.2 (2013): 64-79. Kai Song, <i>Jie Ren,</i> Zhiyuan Zhai, Xuemei Liu, Minghua Deng, and Fengzhu Sun. [<a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2012.0228"><span class="s6">paper</span></a>]</span></li>
  <li class="li6"><b></b><span class="s1"><b>Comparison of Metagenomic Samples Using Sequence Signatures. </b><i>BMC Genomics</i> 13.1 (2012): 1-17.<b> </b>Bai Jiang*, Kai Song*, <i>Jie Ren*</i>, Minghua Deng, Fengzhu Sun, and Xuegong Zhang. [<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-13-730"><span class="s6">paper</span></a>]</span></li>
</ul>
<p class="p6"><span class="s1"> </span></p>
<p class="p5"><span class="s1"><b>Talks</b></span></p>
<ul class="ul1">
  <li class="li1"><span class="s1">2023 <a href="http://wids-santaclara.org/?i=1"><span class="s6">Women in Data Science SANTA CLARA 2023</span></a>: Building neural networks that know what they don't know.</span></li>
  <li class="li1"><span class="s1">2023 <a href="https://sites.google.com/view/vand-cvpr23/home"><span class="s6">Visual Anomaly and Novelty Detection Workshop</span></a> @ CVPR 2023: OOD detection and uncertainty estimation using text-image models.</span></li>
  <li class="li1"><span class="s1">2022 Amazon BioSciences Colloquium: Does Your Model Know What It Doesn’t Know? Exploring the Limits of Out-of-distribution Detection.</span></li>
  <li class="li1"><span class="s1">2022 UCL Centre for Artificial Intelligence: <a href="https://www.youtube.com/watch?v=8cdBWK4Y45s"><span class="s6">Reliable Deep Anomaly Detection</span></a></span></li>
  <li class="li10"><span class="s7"></span><span class="s8">2021 Anomaly Detection for Scientific Discovery Seminars: <a href="https://ad4sd.github.io/seminars/"><span class="s6">Exploring the limits of out-of-distribution detection in vision and biomedical applications</span></a></span></li>
  <li class="li10"><span class="s7"></span><span class="s8">2021 ICML WiML Un-Workshop: <a href="https://docs.google.com/presentation/d/1E90g5yJpYZBa-_jRYGc0IMSnE0axf0wHskUKS1XD-9Y/edit?usp=sharing"><span class="s6">Uncertainty Estimation and Out-of-Distribution (Ood) Detection in Deep Learning</span></a></span></li>
  <li class="li10"><span class="s7"></span><span class="s8">2019 NeurIPS: <a href="https://www.youtube.com/watch?v=-FduW9ZWAR4"><span class="s6">Likelihood Ratios for Out-of-Distribution Detection.</span></a></span></li>
  <li class="li1"><span class="s1">2016 Molecular and Computational Biology Retreat, University of Southern California: Predicting Virus-Host Interactions Using Sequence Signatures.</span></li>
  <li class="li1"><span class="s1">2015 Joint USC/UCLA Bioinformatics Colloquium: Statistical Inference of Markov Models of Genomic Sequences Using NGS Data.</span></li>
  <li class="li1"><span class="s1">2015 Research in Computational Molecular Biology (RECOMB)-Seq: Inference of Markovian Properties of Molecular Sequences From NGS Data and Applications to Comparative Genomics.</span></li>
  <li class="li1"><span class="s1">2013 Molecular and Computational Biology Retreat, University of Southern California: Alignment-Free Genome and Metagenome Comparison Based on Next Generation Sequencing Reads.</span></li>
</ul>
<p class="p6"><span class="s1"> </span></p>
<p class="p5"><span class="s1"><b>Mentorship and Community</b></span></p>
<ul class="ul1">
  <li class="li1"><span class="s1">Interns: Yunhao Ge (2022), James Allingham (2022), Kundan Krishna (2022), Yu-An Chung (2019).</span></li>
  <li class="li1"><span class="s1">Invited reviewer for machine learning venues: NeurIPS (2019-), ICML (2019-), ICLR (2019-), IEEE Transactions on Artificial Intelligence </span></li>
  <li class="li1"><span class="s1">Invited reviewer for bioinformatics journals: Nature Microbiology, Cell Systems, Medical Image Analysis, Nucleic Acids Research, Gut, Bioinformatics, Giga Science, Scientific Reports, PLOS Computational Biolofy, PLOS ONE. </span></li>
  <li class="li1"><span class="s1">Program Committee (PC) for ICML workshop on Uncertainty and Robustness in Deep Learning  (2020-2021)<span class="Apple-converted-space"> </span></span></li>
</ul>
<p class="p6"><span class="s1"> </span></p>
<p class="p5"><span class="s1"><b>Education</b></span></p>
<ul class="ul1">
  <li class="li1"><span class="s1">Postdoc Fellow, University of Southern California, Los Angeles, USA. (2017-2018).</span></li>
  <li class="li1"><span class="s1">Ph.D. in Computational Biology and Bioinformatics, University of Southern California, Los Angeles, USA. (2013-2017).</span></li>
  <ul class="ul2">
    <li class="li10"><span class="s7"></span><span class="s8">Advisors: <a href="https://dornsife.usc.edu/cf/faculty-and-staff/faculty.cfm?pid=1003742"><span class="s6">Fengzhu Sun</span></a>, <a href="https://dornsife.usc.edu/cf/faculty-and-staff/faculty.cfm?pid=1003269"><span class="s6">Jed Fuhrman</span></a>, <a href="https://dornsife.usc.edu/labs/msw/"><span class="s6">Michael Waterman</span></a>, <a href="http://www.stats.ox.ac.uk/~reinert/"><span class="s6">Gesine Reinert</span></a></span></li>
  </ul>
  <li class="li1"><span class="s1">M.S. in Statistics, University of Southern California, Los Angeles, USA. (2016-2017).</span></li>
  <li class="li1"><span class="s1">Exchange Student in Bioinformatics, Stockholm University, Stockholm, Sweden. (2011).</span></li>
  <li class="li1"><span class="s1">M.S. in Probability and Statistics, Peking University, Beijing, China. (2010-2013).</span></li>
  <li class="li1"><span class="s1">Double Major in Economics, Peking University, Beijing, China. (2009-2011).</span></li>
  <li class="li1"><span class="s1">Exchange Student in Mathematics, Hong Kong Baptist University, Hong Kong SAR. (2008).</span></li>
  <li class="li1"><span class="s1">B.S. in Information and Computing Science, Minzu University, Beijing, China. (2006-2010).</span></li>
</ul>
<p class="p6"><span class="s1"> </span></p>
<p class="p6"><span class="s1"> </span></p>

<!-- attempt hiding from spambots :p -->
<script type="text/javascript">
var e_is_shown = false;
document.getElementById('iemail').addEventListener("click", function(){
    let demail = document.getElementById('demail');
    demail.innerHTML = 'jie.ren.usc' + ' _at_ ' + 'gmail' + '.com';
    demail.style.opacity = e_is_shown ? 0 : 1;
    e_is_shown = !e_is_shown;
})
</script>

</body>
</html>
